---
title: "eQTL B6 vs BXD references"
author: "Nastassia Gobet"
date: '`r Sys.Date()`'
output:
  html_document:
    code_folding: hide
    fig_caption: yes
    highlight: tango
    number_sections: no
    theme: readable
    toc: yes
    toc_depth: 4
    df_print: paged
  pdf_document:
    fig_caption: yes
  word_document:
    toc: yes
    toc_depth: '4'
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo=TRUE, fig.path="Graphs/eQTLB6vsBXDreferences/", warning=TRUE)

#load needed libraries
library(knitr)
library(vioplot)

# set working directory
##setwd("F:/BXD/analysis/Mapping")
```

# Goal

Compare eQTLs with BXD-specific references and B6mm10 reference:

* Determine examples of genes eQTLs are the most different between B6 and BXD references.
* Quantify the impact at at eQTL level.

# Information

Doing this in default STAR setting (genotypesandimputed_withannotation_Local_0_10) because also available with B6 mm10 reference (abbreviated B6)

FastQTL output (http://fastqtl.sourceforge.net/):

* rowname = ID molecular phenotype (gene)
* V2 = number of variant tested
* V3 = MLE of shape 1 parameter of beta distribution
* V4 = MLE of shape 2 parameters of beta distribution
* V5 = to be described (not explained on FastQTL website)
* V6 = ID of best variant
* V7 = distance between variant and molecular phenotype
* V8 = p-value of association between variant and molecular phenotype
* V9 = slope of regression
* V10 = first permutation p-value accounting for multiple variants tested by molecular phenotype
* V11 = second permutation p-value obtained with beta approximation (recommended to use this one)

# Preparation

## Functions definitions

```{r FunctionsDefinitions}
prepareRawCounts <- function(counts, ref){
  
  # convert gene ids to gene names
  idx_uniq <- which(!duplicated(GeneConvert$V2[match(rownames(counts), GeneConvert$V1)]))
  counts <- counts[idx_uniq,]
  rownames(counts) <- GeneConvert$V2[match(rownames(counts), GeneConvert$V1)]
  
  # change X into C for cortex
  colnames(counts) <- gsub("X", "C", colnames(counts))
  # split by tissue and condition
  CortexNSD <- subset(counts, select=grep("^C[0-9]*nsd$", colnames(counts), value=TRUE))
  CortexSD <- subset(counts, select=grep("^C[0-9]*sd$", colnames(counts), value=TRUE))
  LiverNSD <- subset(counts, select=grep("^L[0-9]*nsd$", colnames(counts), value=TRUE))
  LiverSD <- subset(counts, select=grep("^L[0-9]*sd$", colnames(counts), value=TRUE))
  # clean and formate names of mouse lines
  colnames(CortexNSD) <- LinesNames$V5[match(gsub("[LCnsd]", "", colnames(CortexNSD)), LinesNames$V10)]
  colnames(CortexSD) <- LinesNames$V5[match(gsub("[LCnsd]", "", colnames(CortexSD)), LinesNames$V10)]
  colnames(LiverNSD) <- LinesNames$V5[match(gsub("[LCnsd]", "", colnames(LiverNSD)), LinesNames$V10)]
  colnames(LiverSD) <- LinesNames$V5[match(gsub("[LCnsd]", "", colnames(LiverSD)), LinesNames$V10)]
  
  # output 4 datasets
  assign(paste0("counts", ref, "ref_CortexNSD"), CortexNSD, envir=parent.frame())
  assign(paste0("counts", ref, "ref_CortexSD"), CortexSD, envir=parent.frame())
  assign(paste0("counts", ref, "ref_LiverNSD"), LiverNSD, envir=parent.frame())
  assign(paste0("counts", ref, "ref_LiverSD"), LiverSD, envir=parent.frame())
}

# find top 3 genes for affected
top3genes <- function(tissue, condition){
  t <- unlist(strsplit(tissue, split=""))[1]
  
  dataB6 <- eval(parse(text=(paste0("dataB6", t, condition))))
  dataB6full <- eval(parse(text=(paste0("dataB6full", t, condition))))
  dataBXD <- eval(parse(text=(paste0("dataBXD", t, condition))))
  dataBXDfull <- eval(parse(text=(paste0("dataBXDfull", t, condition))))

  # identify genes in common between the B6 and BXD lists
  commoneGenes <- intersect(rownames(dataB6), rownames(dataBXD))

  # remove genes with 0 variants
  commoneGenes <- setdiff(commoneGenes, rownames(dataB6full)[dataB6full$V2==0])

  # bigger difference in slope
  orderedgenes <- commoneGenes[order(abs(dataB6full[commoneGenes, "V9"]-dataBXDfull[commoneGenes, "V9"]), decreasing=TRUE)]
  ##dataB6full[match(orderedgenes[1:3], rownames(dataB6full)),]
  ##dataBXDfull[match(orderedgenes[1:3], rownames(dataBXDfull)),]
  outtable <- cbind(dataB6full[orderedgenes[1:3], c(5, 8)], dataB6[orderedgenes[1:3], "adjustedpvalue"], dataBXDfull[orderedgenes[1:3], c(5, 8)], dataBXD[orderedgenes[1:3], "adjustedpvalue"])
  colnames(outtable) <- c("variantB6", "slopeB6", "qvalueB6", "variantBXD", "slopeBXD", "qvalueBXD")
  
  # output a table with info on 3 top affected genes
  assign(paste0("top3genes", t, condition), outtable, envir=parent.frame())
}

# plot gene expression for a pair gene-variant
plotGeneExpressionVariant <- function(ref, gene, variant, tissue, condition){
  # get first letter of tissue
   t <- unlist(strsplit(tissue, split=""))[1]
  # get gene expression dataset
  expr <- eval(parse(text=paste0("expr", ref, "ref", t, condition)))
  # plot
  plot(as.numeric(expr[gene, sort(names(expr))]), pch=19, col=as.factor(genotypesGN[genotypesGN$Locus==variant, sort(names(expr))]), main=paste0("Gene: ", gene,", ref: ", ref, ", variant: ", variant, ", tissue: ", tissue, ", condition: ", condition), xaxt="n", xlab="", ylab="gene expression", las=1)
  axis(1, 1:length(names(expr)), sort(names(expr)), las=2)
}
# vectorize function
plotGeneExpressionVariant <- Vectorize(plotGeneExpressionVariant, SIMPLIFY=FALSE)

# plot raw gene counts for a pair gene-variant
plotGeneCountsVariant <- function(ref, gene, variant, tissue, condition){
  # get gene expression dataset
  counts <- eval(parse(text=paste0("counts", ref, "ref_", tissue, condition)))
  # plot
  plot(as.numeric(counts[gene, sort(names(counts))]), pch=19, col=as.factor(genotypesGN[genotypesGN$Locus==variant, sort(names(counts))]), main=paste0("Gene: ", gene,", ref: ", ref, ", variant: ", variant, ", tissue: ", tissue, ", condition: ", condition), xaxt="n", xlab="", ylab="raw gene counts", las=1)
  axis(1, 1:length(names(counts)), sort(names(counts)), las=2)
}
# vectorize function
plotGeneCountsVariant <- Vectorize(plotGeneCountsVariant, SIMPLIFY=FALSE)
```

## Data preparation

```{r DataPreparation}
# load table to convert mouse line names
LinesNames <- read.table("F:/BXD/data/ConvertLineNames.tsv", header=FALSE, stringsAsFactors=FALSE, sep="\t")

# load table to convert between gene id and gene name
GeneConvert <- read.table("F:/BXD/references/gene_names_B6mm10.tab", header=FALSE, stringsAsFactors=FALSE, sep="\t")

# retrieve GeneNetwork (GN) genotypes
genotypesGN <- read.table("F:/BXD/data/genome/BXDGenotypes.geno", header=TRUE)

# retrieve eQTL results
for(tissue in c("Cortex", "Liver")){
  t <- unlist(strsplit(tissue, split=""))[1]
  for(condition in c("NSD", "SD")){
    assign(paste0("dataB6", t, condition), read.table(paste0("F:/BXD/data/transcriptome/eQTL_B6mm10primaryassembly_withannotation/TMMnormalized_log2CPM_", tissue, "_", condition, "_pvalcorrected.txt"), row.names=1, stringsAsFactors=FALSE))
    assign(paste0("dataB6full", t, condition), read.table(paste0("F:/BXD/data/transcriptome/eQTL_B6mm10primaryassembly_withannotation/TMMnormalized_log2CPM_", tissue, "_", condition, ".txt"), row.names=1, stringsAsFactors=FALSE))
    assign(paste0("dataBXD", t, condition), read.table(paste0("F:/BXD/data/transcriptome/eQTL_BXDfromGNandimputedD2blocks_withannotation/TMMnormalized_log2CPM_", tissue, "_", condition,"_pvalcorrected.txt"), row.names=1, stringsAsFactors=FALSE))
    assign(paste0("dataBXDfull", t, condition), read.table(paste0("F:/BXD/data/transcriptome/eQTL_BXDfromGNandimputedD2blocks_withannotation/TMMnormalized_log2CPM_", tissue, "_", condition, ".txt"), row.names=1, stringsAsFactors=FALSE))
  }
}

# retrieve raw gene counts, formate and subset
countsB6ref <- read.table("F:/BXD/data/transcriptome/2_mapping_STAR/OnGenomeAndAnnotation/B6mm10primaryassembly_withannotation/Summary_ReadsPerGene.out.tab", header=TRUE, row.names=1)
prepareRawCounts(countsB6ref, "B6")
countsBXDref <- read.table("F:/BXD/data/transcriptome/2_mapping_STAR/OnGenomeAndAnnotation/BXDfromGNandimputedD2blocks_withannotation/Summary_ReadsPerGene.out.tab", header=TRUE, row.names=1)
prepareRawCounts(countsBXDref, "BXD")

# retrieve gene expression
exprB6refCNSD <- read.table("F:/BXD/data/transcriptome/2_mapping_STAR/OnGenomeAndAnnotation/B6mm10primaryassembly_withannotation/TMMnormalized_log2CPM_Cortex_NSD.tab")
exprBXDrefCNSD <- read.table("F:/BXD/data/transcriptome/2_mapping_STAR/OnGenomeAndAnnotation/BXDfromGNandimputedD2blocks_withannotation/TMMnormalized_log2CPM_Cortex_NSD.tab")
exprB6refCSD <- read.table("F:/BXD/data/transcriptome/2_mapping_STAR/OnGenomeAndAnnotation/B6mm10primaryassembly_withannotation/TMMnormalized_log2CPM_Cortex_SD.tab")
exprBXDrefCSD <- read.table("F:/BXD/data/transcriptome/2_mapping_STAR/OnGenomeAndAnnotation/BXDfromGNandimputedD2blocks_withannotation/TMMnormalized_log2CPM_Cortex_SD.tab")
exprB6refLNSD <- read.table("F:/BXD/data/transcriptome/2_mapping_STAR/OnGenomeAndAnnotation/B6mm10primaryassembly_withannotation/TMMnormalized_log2CPM_Liver_NSD.tab")
exprBXDrefLNSD <- read.table("F:/BXD/data/transcriptome/2_mapping_STAR/OnGenomeAndAnnotation/BXDfromGNandimputedD2blocks_withannotation/TMMnormalized_log2CPM_Liver_NSD.tab")
exprB6refLSD <- read.table("F:/BXD/data/transcriptome/2_mapping_STAR/OnGenomeAndAnnotation/B6mm10primaryassembly_withannotation/TMMnormalized_log2CPM_Liver_SD.tab")
exprBXDrefLSD <- read.table("F:/BXD/data/transcriptome/2_mapping_STAR/OnGenomeAndAnnotation/BXDfromGNandimputedD2blocks_withannotation/TMMnormalized_log2CPM_Liver_SD.tab")
```

# Quantifying

```{r echo=FALSE}
read_chunk('F:/BXD/analysis/scripts/mapping/compareeQTLB6toBXD.R')
```

## Cortex NSD

```{r quantifyCortexNSD}
tissue <- "Cortex"
condition <- "NSD"

<<joineQTLdatasets>>

<<quantify>>

<<qvalue_package_diagnostic>>

```

## Cortex SD

```{r quantifyCortexSD}
tissue <- "Cortex"
condition <- "SD"

<<joineQTLdatasets>>

<<quantify>>

<<qvalue_package_diagnostic>>

```

## Liver NSD

```{r quantifyLiverNSD}
tissue <- "Liver"
condition <- "NSD"

<<joineQTLdatasets>>

<<quantify>>

<<qvalue_package_diagnostic>>

```

## Liver SD

```{r quantifyLiverSD}
tissue <- "Liver"
condition <- "SD"

<<joineQTLdatasets>>

<<quantify>>

<<qvalue_package_diagnostic>>

```

## compare with Data descriptor

```{r}
# load mm9 cis-eQTL
ceLNSD_mm9 <- read.table("F:/BXD/BACKUP/mm9/Data/IntermediateLayer/cis-eQTL/Liver_nsd/ciseQTL.Liver.NSD.pvalcorrected.txt",sep='')
ceLSD_mm9 <- read.table("F:/BXD/BACKUP/mm9/Data/IntermediateLayer/cis-eQTL/Liver_sd/ciseQTL.Liver.SD.pvalcorrected.txt",sep='')
ceCNSD_mm9 <- read.table("F:/BXD/BACKUP/mm9/Data/IntermediateLayer/cis-eQTL/Cortex_nsd/ciseQTL.Cortex.NSD.pvalcorrected.txt",sep='')
ceCSD_mm9 <- read.table("F:/BXD/BACKUP/mm9/Data/IntermediateLayer/cis-eQTL/Cortex_sd/ciseQTL.Cortex.SD.pvalcorrected.txt",sep='')
# load mm10 cis-eQTL
ceLNSD_mm10 <- read.table("F:/BXD/BACKUP/mm10/Data/IntermediateLayer/cis-eQTL/ciseQTL.Liver.NSD.pvalcorrected.txt",sep='')
ceLSD_mm10 <- read.table("F:/BXD/BACKUP/mm10/Data/IntermediateLayer/cis-eQTL/ciseQTL.Liver.SD.pvalcorrected.txt",sep='')
ceCNSD_mm10 <- read.table("F:/BXD/BACKUP/mm10/Data/IntermediateLayer/cis-eQTL/ciseQTL.Cortex.NSD.pvalcorrected.txt",sep='')
ceCSD_mm10 <- read.table("F:/BXD/BACKUP/mm10/Data/IntermediateLayer/cis-eQTL/ciseQTL.Cortex.SD.pvalcorrected.txt",sep='')

for(data in c("ceCNSD_mm9", "ceCSD_mm9", "ceLNSD_mm9", "ceLSD_mm9", "ceCNSD_mm10", "ceCSD_mm10", "ceLNSD_mm10", "ceLSD_mm10")){
  d <- get(data)
  print(data)
  print(nrow(d))
  print(nrow(d[d$adjustedpvalue<0.05,]))
  print(nrow(d[d$adjustedpvalue<0.05,]) / nrow(d) * 100)
}

```


# Examples of genes

```{r}
# define color palette
palette(c("black", "tan", "red"))
```


## Legend

* black = B6 allele
* tan = D2 allele
* red = heterozygous

## Cortex NSD

```{r CortexNSD, fig.width=10}
# find and display information on 
top3genes(tissue="Cortex", condition="NSD")
top3genesCNSD

# plot variant-gene expression pair
plotGeneExpressionVariant(ref="B6", gene=rownames(top3genesCNSD), variant=top3genesCNSD$variantB6, tissue="Cortex", condition="NSD")
plotGeneExpressionVariant(ref="BXD", gene=rownames(top3genesCNSD), variant=top3genesCNSD$variantBXD, tissue="Cortex", condition="NSD")
  
# plot variant-gene count pair
plotGeneCountsVariant(ref="B6", gene=rownames(top3genesCNSD), variant=top3genesCNSD$variantB6, tissue="Cortex", condition="NSD")
plotGeneCountsVariant(ref="BXD", gene=rownames(top3genesCNSD), variant=top3genesCNSD$variantBXD, tissue="Cortex", condition="NSD")
```


## Cortex SD

```{r CortexSD, fig.width=10}
# find and display information on 
top3genes(tissue="Cortex", condition="SD")
top3genesCSD

# plot variant-gene expression pair
plotGeneExpressionVariant(ref="B6", gene=rownames(top3genesCSD), variant=top3genesCSD$variantB6, tissue="Cortex", condition="SD")
plotGeneExpressionVariant(ref="BXD", gene=rownames(top3genesCSD), variant=top3genesCSD$variantBXD, tissue="Cortex", condition="SD")
  
# plot variant-gene count pair
plotGeneCountsVariant(ref="B6", gene=rownames(top3genesCSD), variant=top3genesCSD$variantB6, tissue="Cortex", condition="SD")
plotGeneCountsVariant(ref="BXD", gene=rownames(top3genesCSD), variant=top3genesCSD$variantBXD, tissue="Cortex", condition="SD")
```


```{r CSDcommon, eval=FALSE, include=FALSE}

# how many genes have a difference in qvalue higher than 0.1?
length(which(abs(qvalueB6-qvalueBXD)>0.1))

# how many genes are under 0.05 with B6 reference and over with BXD references?
length(which(qvalueB6<0.05&qvalueBXD>0.05))

# how many genes are over 0.05 with B6 reference and under with BXD references?
length(which(qvalueB6>0.05&qvalueBXD<0.05))

# how many genes are over or under depending on the reference?
length(which(qvalueB6<0.05&qvalueBXD>0.05))+length(which(qvalueB6>0.05&qvalueBXD<0.05))
```

About 2% of the genes are highly affected. Knowing we affected less than 1% of bp in the genome.

legend: black=B6 allele, tan=D2 allele

Checking for raw gene counts. There are comparable since we used the same reads on B6 or BXD references.

```{r CSDcounts, eval=FALSE, include=FALSE}
# retrieve raw gene counts, formate and subset cortex SD values
countsB6ref <- read.table("F:/BXD/data/transcriptome/2_mapping_STAR/OnGenomeAndAnnotation/B6mm10primaryassembly_withannotation/Summary_ReadsPerGene.out.tab", header=TRUE, row.names=1)
colnames(countsB6ref) <- gsub("X", "C", colnames(countsB6ref))
countsB6ref_CortexSD <- subset(countsB6ref, select=grep("^C[0-9]*sd$", colnames(countsB6ref), value=TRUE))
colnames(countsB6ref_CortexSD) <- gsub("[Csd]", "", colnames(countsB6ref_CortexSD))
colnames(countsB6ref_CortexSD) <- LinesNames$V5[match(colnames(countsB6ref_CortexSD), LinesNames$V10)]

countsBXDref <- read.table("F:/BXD/data/transcriptome/2_mapping_STAR/OnGenomeAndAnnotation/BXDfromGNandimputedD2blocks_withannotation/Summary_ReadsPerGene.out.tab", header=TRUE, row.names=1)
colnames(countsBXDref) <- gsub("X", "C", colnames(countsBXDref))
countsBXDref_CortexSD <- subset(countsBXDref, select=grep("^C[0-9]*sd$", colnames(countsBXDref), value=TRUE))
colnames(countsBXDref_CortexSD) <- gsub("[Csd]", "", colnames(countsBXDref_CortexSD))
colnames(countsBXDref_CortexSD) <- LinesNames$V5[match(colnames(countsBXDref_CortexSD), LinesNames$V10)]

# load table to convert between gene id and gene name
GeneConvert <- read.table("F:/BXD/references/gene_names_B6mm10.tab", header=FALSE, stringsAsFactors=FALSE, sep="\t")
# convert gene names into gene ids
top3genesid <- GeneConvert$V1[match(top3genes, GeneConvert$V2)]

# Myh6
plot(as.numeric(countsBXDref_CortexSD["ENSMUSG00000040752", sort(names(countsBXDref_CortexSD))]), pch=19, col=as.factor(genotypesGN[genotypesGN$Locus=="rs30746379", sort(names(countsBXDref_CortexSD))]), main="Gene: Myh6, ref: BXD, variant: rs30746379", xaxt="n", xlab="", ylab="raw count")
axis(1, 1:length(names(countsBXDref_CortexSD)), sort(names(countsBXDref_CortexSD)), las=2)

plot(as.numeric(countsB6ref_CortexSD["ENSMUSG00000040752", sort(names(countsB6ref_CortexSD))]), pch=19, col=as.factor(genotypesGN[genotypesGN$Locus=="rs3713576", sort(names(countsB6ref_CortexSD))]), main="Gene: Myh6, ref: B6, variant: rs3713576", xaxt="n", xlab="", ylab="raw count")
axis(1, 1:length(names(countsB6ref_CortexSD)), sort(names(countsB6ref_CortexSD)), las=2)

# Gm4366
plot(as.numeric(countsBXDref_CortexSD["ENSMUSG00000107383", sort(names(countsBXDref_CortexSD))]), pch=19, col=as.factor(genotypesGN[genotypesGN$Locus=="rs31882225", sort(names(countsBXDref_CortexSD))]), main="Gene: Gm4366, ref: BXD, variant: rs31882225", xaxt="n", xlab="", ylab="raw count")
axis(1, 1:length(names(countsBXDref_CortexSD)), sort(names(countsBXDref_CortexSD)), las=2)

plot(as.numeric(countsB6ref_CortexSD["ENSMUSG00000107383", sort(names(countsB6ref_CortexSD))]), pch=19, col=as.factor(genotypesGN[genotypesGN$Locus=="rs32413905", sort(names(countsB6ref_CortexSD))]), main="Gene: Gm4366, ref: B6, variant: rs32413905", xaxt="n", xlab="", ylab="raw count")
axis(1, 1:length(names(countsB6ref_CortexSD)), sort(names(countsB6ref_CortexSD)), las=2)

# Gm5499
plot(as.numeric(countsBXDref_CortexSD["ENSMUSG00000071035", sort(names(countsBXDref_CortexSD))]), pch=19, col=as.factor(genotypesGN[genotypesGN$Locus=="rs33397913", sort(names(countsBXDref_CortexSD))]), main="Gene: Gm5499, ref: BXD, variant: rs33397913", xaxt="n", xlab="", ylab="raw count")
axis(1, 1:length(names(countsBXDref_CortexSD)), sort(names(countsBXDref_CortexSD)), las=2)

plot(as.numeric(countsB6ref_CortexSD["ENSMUSG00000071035", sort(names(countsB6ref_CortexSD))]), pch=19, col=as.factor(genotypesGN[genotypesGN$Locus=="rs33397913", sort(names(countsB6ref_CortexSD))]), main="Gene: Gm5499, ref: B6, variant: rs33397913", xaxt="n", xlab="", ylab="raw count")
axis(1, 1:length(names(countsB6ref_CortexSD)), sort(names(countsB6ref_CortexSD)), las=2)
```

## Liver NSD

```{r LiverNSD, fig.width=10}
# find and display information on 
top3genes(tissue="Liver", condition="NSD")
top3genesLNSD

# plot variant-gene expression pair
plotGeneExpressionVariant(ref="B6", gene=rownames(top3genesLNSD), variant=top3genesLNSD$variantB6, tissue="Liver", condition="NSD")
plotGeneExpressionVariant(ref="BXD", gene=rownames(top3genesLNSD), variant=top3genesLNSD$variantBXD, tissue="Liver", condition="NSD")
  
# plot variant-gene count pair
plotGeneCountsVariant(ref="B6", gene=rownames(top3genesLNSD), variant=top3genesLNSD$variantB6, tissue="Liver", condition="NSD")
plotGeneCountsVariant(ref="BXD", gene=rownames(top3genesLNSD), variant=top3genesLNSD$variantBXD, tissue="Liver", condition="NSD")
```

## Liver SD

```{r LiverSD, fig.width=10}
# find and display information on 
top3genes(tissue="Liver", condition="SD")
top3genesLSD

# plot variant-gene expression pair
plotGeneExpressionVariant(ref="B6", gene=rownames(top3genesLSD), variant=top3genesLSD$variantB6, tissue="Liver", condition="SD")
plotGeneExpressionVariant(ref="BXD", gene=rownames(top3genesLSD), variant=top3genesLSD$variantBXD, tissue="Liver", condition="SD")
  
# plot variant-gene count pair
plotGeneCountsVariant(ref="B6", gene=rownames(top3genesLSD), variant=top3genesLSD$variantB6, tissue="Liver", condition="SD")
plotGeneCountsVariant(ref="BXD", gene=rownames(top3genesLSD), variant=top3genesLSD$variantBXD, tissue="Liver", condition="SD")
```


# Session information

```{r}
sessionInfo()
```
