---
title: "Figures BXD mapping paper"
author: "Nastassia Gobet"
date: '`r Sys.Date()`'
output:
  html_document:
    code_folding: hide
    df_print: paged
    fig_caption: yes
    highlight: tango
    number_sections: no
    theme: readable
    toc: yes
    toc_depth: 4
  pdf_document:
    fig_caption: yes
  word_document:
    toc: yes
    toc_depth: '4'
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(fig.path="A:/CIG/USERS/ngobet/Communication/Figures/Mapping_paper/", warning=TRUE, echo=FALSE)

#load libraries
library(knitr)
library(VennDiagram)
library(gridBase)
library(magick)
library(plotrix)

# set working directory
##setwd("F:/BXD/analysis/Mapping")
```

# Goal

The goal is to produce the figures for the BXD mapping paper.

# Figures

## Figure 1: Including genomic variants in inbred mouse transcriptome read mapping overview.

```{r Figure1_RNAseqReadMappingStrategies}
# A
# get BXD panel image
BXDpanel <- image_read_svg("A:/CIG/USERS/ngobet/Communication/Figures/BXDlineage_20200831.svg", height=500)
# add letter for panel
fig1a <- image_annotate(BXDpanel, "A", size=36, location="+0+0", color="black")

# B
# get simplified RNA-seq workflow 
BXDmappingstrategy <- image_read_svg("A:/CIG/USERS/ngobet/Communication/Figures/Mapping_paper/BXDmappingstrategy_20210621.svg", height=500)
# add letter for panel
fig1b <- image_annotate(BXDmappingstrategy, "B", size=36, location="+0+0", color="black")

# C
# number of variants per category
variants <- c(7324, 5872394, 1093496, 1689181)
var_categories <- c("genotypes", "SNVs", "indels", "SVs")
variantsandgeno <- image_graph(width=500, height=350, res=96)
par(mar=c(3.1, 5.6, 0.1, 0.1), mgp=c(4,1,0))
barsPosition <- barplot(variants, names.arg=var_categories, las=1, ylim=c(0,6500000), col="white", ylab="Number of variants", cex.lab=1, cex.names=1)
yoffset <- 3e5
text(barsPosition, variants+yoffset, labels=variants, adj=0.5, xpd=TRUE, col="grey20", cex=0.8)
d1 <- dev.off()
fig1c <- image_annotate(variantsandgeno, "C", size=28, location="+0+0", color="black")

# D
imputation <- image_read_svg("A:/CIG/USERS/ngobet/Communication/Figures/Mapping_paper/GenotypesImputation_20210621.svg", height=345)
# add letter label
fig1d <- image_annotate(imputation, "D", size=36, location="+0+0", color="black")

# display full figure
toppanels <- image_append(c(fig1a, fig1b))
bottompanels <- image_append(c(fig1c, fig1d))
image_append(c(toppanels, bottompanels), stack=TRUE)
```

__Figure 1 | Including genomic variants in inbred mouse transcriptome read mapping overview.__

A) BXD mouse recombinant inbred panel. Samples come from mice that are: BXD advanced recombinant inbred lines, parental inbred strains which are C57BL/6J (B6) and DBA/2J (D2), and not inbred first generation cross between the parental strains (F1).
B) RNA-seq read mapping strategies in this study.
C) BXD genotypes available from GeneNetwork and D2-specific genomic variants available from dbSNP.
D) Genotypes imputation workflow. B6 regions or alleles are in black, D2 regions or alleles are in brown.


## Figure 2: Two parental assemblies strategy.

```{r prepare_Figure2}
# load mapping statistics on genome + transcriptome annotation (full GRC, not primary assembly)
map_B6mm10 <- read.table("F:/BXD/data/transcriptome/2_Mapping_STAR/OnGenomeAndAnnotation/MappingStatisticsB6mm10primaryassembly.tsv", header=TRUE, stringsAsFactors=FALSE, row.names="Sample_Name")
map_D2 <- read.table("F:/BXD/data/transcriptome/2_Mapping_STAR/OnGenomeAndAnnotation/MappingStatisticsD2assembly.tsv", header=TRUE, stringsAsFactors=FALSE, row.names="Sample_Name")

# Load data mapped on genome
B6mm10allG <- read.table("F:/BXD/data/transcriptome/2_mapping_STAR/OnGenome/MappingStatisticsB6mm10_primaryassembly.tsv", header=TRUE, stringsAsFactors=FALSE, row.names="Sample_Name")
D2allG <- read.table("F:/BXD/data/transcriptome/2_mapping_STAR/OnGenome/MappingStatisticsD2.tsv", header=TRUE, stringsAsFactors=FALSE, row.names="Sample_Name")

# Process data
# get parental and F1
parentalF1samples_idx <- rownames(D2allG)[grep("[DB]", rownames(D2allG))]

# load table with results of differential expression
DMdata_Cortex <- read.table('F:/BXD/data/transcriptome/5_DifferentialMapping/DifferentialMapping_Limma_Cortex_20190917.txt', sep="\t")
DMdata_Liver <- read.table('F:/BXD/data/transcriptome/5_DifferentialMapping/DifferentialMapping_Limma_Liver_20190917.txt', sep="\t")
```

```{r Figure2_TwoParentalAssembliesStrategy, fig.width=7, fig.height=5}
# prepare figures with 2 parental assembly mapping
layout(matrix(c(1,1,2,3,4,5), nrow=2, ncol=3, byrow=TRUE))
par(bty="l", col.lab="grey30", col.axis="grey30", col.main="grey30", fg="grey30")

# A
plot(map_B6mm10$Uniq, pch=18, col="black", ylim=c(0,100), xaxt="n",
     main="Mapping on genome (+ annotation) with STAR",
     xlab="", ylab="Uniquely mapped reads (%)",
     las=1)
points(map_D2$Uniq, pch=20, col="tan")
for(myline in seq(0,2,0.5)){
        axis(1, at=88, line=myline, labels=c(""))
}
axis(1, at=c(43,129), line=0, tick=FALSE, labels=c("cortex", "liver"))
legend("bottomleft", title="Reference assembly", legend=c("GRCm38","D2"), col=c("black","tan"), pch=c(18,20), cex=0.9)

# B
# mapping on genome
plot(B6mm10allG[parentalF1samples_idx,"Uniq0"], pch=18, col="black", ylim=c(0,100), las=2, xaxt="n", ylab="Uniquely mapped percentage", xlab="", main="0 mismatch allowed on genome", cex.main=0.9)
points(D2allG[parentalF1samples_idx,"Uniq0"], pch=20, col="tan")
abline(v=c(4.5,6.5,10.5,12.5,16.5,18.5,22.5))
axis(1, at=c(4.5,6.5,10.5,12.5,16.5,18.5,22.5), labels=rep(c(""),7))
axis(1, at=c(c(2.5,5.5,8.5,11.5),12+c(2.5,5.5,8.5,11.5)), tick=FALSE, labels=rep(c("B6","BxD","D2","DxB"),2), las=2)
for(myline in seq(0.5,3.5,0.5)){
  axis(1, at=12.5, line=myline, labels=c(""))
}
axis(1, at=c(6.5,18.5), line=2, tick=FALSE, labels=c("cortex", "liver"))

# C
# mapping on genome
plot(B6mm10allG[parentalF1samples_idx,"Uniq10"], pch=18, col="black", ylim=c(0,100), las=2, xaxt="n", ylab="Uniquely mapped percentage", xlab="", main="10 mismatches allowed on genome", cex.main=0.9)
points(D2allG[parentalF1samples_idx,"Uniq10"], pch=20, col="tan")
abline(v=c(4.5,6.5,10.5,12.5,16.5,18.5,22.5))
axis(1, at=c(4.5,6.5,10.5,12.5,16.5,18.5,22.5), labels=rep(c(""),7))
axis(1, at=c(c(2.5,5.5,8.5,11.5),12+c(2.5,5.5,8.5,11.5)), tick=FALSE, labels=rep(c("B6","BxD","D2","DxB"),2), las=2)
for(myline in seq(0.5,3.5,0.5)){
  axis(1, at=12.5, line=myline, labels=c(""))
}
axis(1, at=c(6.5,18.5), line=2, tick=FALSE, labels=c("cortex", "liver"))

# panel D left
# plot for Cortex
palette(c("black", "red3"))
plot(1:nrow(DMdata_Cortex), DMdata_Cortex$logFC, las=2, pch=20, col=as.factor(DMdata_Cortex$adj.P.Val<=0.05), cex.main=0.9,
       main="DM in Cortex (GRCm38 vs D2)", xlab="rank of adjusted p-value", ylab="-logFC")

# panel D right
# plot for Liver
plot(1:nrow(DMdata_Liver), DMdata_Liver$logFC, las=2, pch=20, 
     col=as.factor(DMdata_Liver$adj.P.Val<=0.05), cex.main=0.9,
       main="DM in Liver (GRCm38 vs D2)", xlab="rank of adjusted p-value", ylab="-logFC")

# add panel labels (letters)
mtext(LETTERS[1:4], side=3, line=0, adj=c(0,0.70,0,0.34), padj=c(1.5,1.5,15,15), cex=1.5, outer=TRUE)
```

__Figure 2 | Two parental assemblies strategy.__

A) Mappability of all samples on 2 parental assemblies (samples are mapped on GRCm38: black symbols and on D2 assembly: brown symbols) using permissive mapping setting (STAR default) in cortex (left) and liver (right).
B) Mappability of parental and F1 (BxD and DxB) samples on 2 parental assemblies using restrictive mapping setting allowing 0 mismatches. Same legend than in A.
C) Mappability of parental and F1 samples on 2 parental assemblies using restrictive mapping setting but allowing up to 10 mismatches. Same legend than in A.
D) Differential mapping (DM) analysis of D2 assembly compared to GRCm38 in the cortex (left) or in the liver (right). Genes are classified as DM genes if FDR adjusted p-value < 0.05 (red) or non DM genes otherwise (black).


## Figure 3: Line-specific references strategy.

```{r prepare_Figure3}
# source function
source("F:/BXD/analysis/scripts/mapping/retrieveSTARstatistics.R")

path <- "F:/BXD/data/transcriptome/2_Mapping_STAR/OnPersonalizedRef/"
statsName <- "Uniquely mapped reads % |"
references <- dir(path, pattern="DBA_2J_genome_")
ref <- "DBA_2J_genome_nonrandomized_indelsSNVs_paternal"
samples <- gsub("_Log.final.out","", list.files(paste0(path, "/", ref), pattern="_Log.final.out"))
resUniqP <- outer(FUN=retrieveStats, X=references, Y=paste0(samples, "_"), path="F:/BXD/data/transcriptome/2_Mapping_STAR/OnPersonalizedRef/", statsName="Uniquely mapped reads % |")
rownames(resUniqP) <- references
colnames(resUniqP) <- samples
baseline <- retrieveStats("B6mm10_primaryassembly", paste0(samples, "_exactunique"), "F:/BXD/data/transcriptome/2_Mapping_STAR/OnGenome", statsName)
resUniqP <- rbind(resUniqP, baseline)
refer <- "DBA_2J_genome_nonrandomized_indelsSNVs_paternal"

limit <- ceiling(max(abs(sweep(resUniqP, MARGIN=2, STATS=resUniqP["baseline",], FUN="-"))))

path <- "F:/BXD/data/MappingEvaluation"
statsName <- "Uniquely mapped reads % |"
lines <- read.table("F:/BXD/data/lines.txt", stringsAsFactors=FALSE)$V1
# remove line 63
lines <- setdiff(lines, "BXD63")
# convert to new nomenclature
LinesNames <- read.table("F:/BXD/data/ConvertLineNames.tsv", header=TRUE, stringsAsFactors=FALSE, sep="\t")
linesNew <- LinesNames$V9[match(lines, LinesNames$V6)]
linesNew <- sort(linesNew, index.return=TRUE)
linesNew$x <- c(linesNew$x[c(-1,-2)], linesNew$x[c(1,2)])
linesNew$ix <- c(linesNew$ix[c(-1,-2)], linesNew$ix[c(1,2)])

# list samples
samples <- c()
getSamplesNames <- function(x,y){paste0(x,linenumber,y)}
for(linenumber in sub("BXD","",lines)){
  samples <- c(samples, c(t(outer(c("","L"),c("nsd","sd"),getSamplesNames))))
}

# retrieve uniquely mapped reads percentage on genotypesandimputed-based customized references
ref_lines <- c(sapply(lines,rep,4))
reference <- "genotypesandimputed_withoutannotation_EndToEnd_1_0"
conditions <- as.factor(rep(c("nsd","sd"),length(samples)/2))
tissues <- as.factor(rep(sort(rep(c("C","L"),2)),length(samples)/4))
genotypesandimputed <- retrieveStats(reference, paste0(samples, "_"), path, statsName)

# retrieve uniquely mapped reads percentage on genotypes-based customized references
path <- "F:/BXD/data/transcriptome/2_Mapping_STAR/OnPersonalizedRef"
references <- paste0(ref_lines, "_genome_nonrandomized_genotypes_paternal")
genotypes <- retrieveStats(references, paste0(samples, "_"), path, statsName)

# retrieve uniquely mapped reads percentage on GRCm38 primary assembly
path <- "F:/BXD/data/transcriptome/2_Mapping_STAR/OnGenome/"
references <- "B6mm10_primaryassembly"
B6mm10 <- retrieveStats(references, paste0(samples, "_exactunique"), path, statsName)

# assemble dataset
geno_restrictive <- data.frame(samples, ref_lines, tissues, conditions, genotypesandimputed, genotypes, B6mm10)


# load table with results of differential expression
DMdataBXD_Cortex <- read.table('F:/BXD/data/transcriptome/5_DifferentialMapping/DifferentialMappingBXDvsB6mm10permissive_Limma_Cortex.txt', sep="\t")
DMdataBXD_Liver <- read.table('F:/BXD/data/transcriptome/5_DifferentialMapping/DifferentialMappingBXDvsB6mm10permissive_Limma_Liver.txt', sep="\t")
```

For GRCm38 vs D2 assembly, `r round(length(which(DMdata_Cortex$adj.P.Val<=0.05))/length(DMdata_Cortex$adj.P.Val)*100)`% of genes are significantly (below or equal to 0.05) differentially mapped in cortex, and `r round(length(which(DMdata_Liver$adj.P.Val<=0.05))/length(DMdata_Liver$adj.P.Val)*100)`% in liver.

For GRCm38 vs BXD-specific, `r round(length(which(DMdataBXD_Cortex$adj.P.Val<=0.05))/length(DMdata_Cortex$adj.P.Val)*100)`% of genes are significantly (below or equal to 0.05) differentially mapped in cortex, and `r round(length(which(DMdataBXD_Liver$adj.P.Val<=0.05))/length(DMdata_Liver$adj.P.Val)*100)`% in liver.

```{r Figure3_Line-specificReferencesStrategy, fig.height=5, fig.width=7}
layout(matrix(c(1,2,3,4,4,4), nrow=2, ncol=3, byrow=TRUE))
par(bty="l", col.lab="grey30", col.axis="grey30", col.main="grey30", fg="grey30")

# panel A
palette(c("red3", "black", "blue3"))
# set sample names
samplesnames <- gsub(pattern="nsd|L",replacement="", names(resUniqP["baseline",]))
samplesnames <- gsub(pattern="^DB$", replacement="DxB", samplesnames)
samplesnames <- gsub(pattern="^BD$", replacement="BxD", samplesnames)
samplesnames <- gsub(pattern="1|2", replacement="", samplesnames)
samplesnames <- gsub(pattern="DB", replacement="D2", samplesnames)
# plot mappability gain
barplot(resUniqP[refer,]-resUniqP["baseline",], col=rep(c("black", "black", "white", "tan", "tan", "white"),2),
          main="Mappability change", las=2, names=samplesnames, cex.names=1, ylab="gain in uniquely mapped reads %", ylim=c(-limit,limit))
abline(h=0, col="black")
abline(v=c(7.3))
axis(3, at=c(3,11), labels=c("Cortex", "Liver"), line=-1, tick=FALSE)

# panel B left
palette(c("black", "red3"))
plot(1:nrow(DMdataBXD_Cortex), DMdataBXD_Cortex$logFC, las=1, pch=20, col=as.factor(DMdataBXD_Cortex$adj.P.Val<=0.05),
       main="DM in Cortex (GRCm38 vs BXD)", xlab="rank of adjusted p-value", ylab="-logFC")

# panel B right
plot(1:nrow(DMdataBXD_Liver), DMdataBXD_Liver$logFC, las=1, pch=20, col=as.factor(DMdataBXD_Liver$adj.P.Val<=0.05),
       main="DM in Liver (GRCm38 vs BXD)", xlab="rank of adjusted p-value", ylab="-logFC")

# panel C
palette(c("lightskyblue", "darkblue", "coral1", "darkred"))
plot(as.vector(matrix(genotypesandimputed, nrow=4)[,linesNew$ix]-matrix(B6mm10, nrow=4)[,linesNew$ix]), type="h", col=1:4, lwd=4, 
     ylim=c(0,6), yaxs="i", xaxt="n", 
     main="", ylab="gain in uniquely mapped reads %", xlab="", bty="l", las=1)
legend("topright", c("Cortex_NSD", "Cortex_SD", "Liver_NSD", "Liver_SD"), fill=1:4, cex=0.7, horiz=TRUE)
axis(1, seq.int(2.5,length(genotypesandimputed),4), linesNew$x, tick=FALSE, las=2, cex.axis=0.8)

# add panel labels (letters)
mtext(LETTERS[1:3], side=3, line=0, adj=c(0,0.34,0), padj=c(1.5,1.5,15), cex=1.5, outer=TRUE)
```

__Figure 3 | Line-specific references strategy.__

A) Relative mappability of customized D2-specific reference (GRCm38 modified with D2-specific indels and SNVs from dbSNP) compared to GRCm38 on parental and F1 samples with exact matches. Samples are all NSD. Colors indicate genetic of the samples: B6 (black), D2 (light brown), and F1 (white) between B6 and D2 strains. The F1 samples are BxD if the mother is B6 and the father is D2 (as for the BXD lines), or the reverse for DxB.
B) Differential mapping (DM) analysis of customized BXD specific references compared to GRCm38, in the cortex (left) or in the liver (right). Genes are classified as DM genes if FDR adjusted p-value < 0.05 (red) or non DM genes otherwise (black).
C) Relative mappability of BXD-specific references (GRCm38 modified for each BXD line with GeneNetwork genotypes and imputed variants) compared to GRCm38 on BXD samples with exact matches.

## Figure 4: Consequences of mapping reference at local eQTL level.

```{r prepare_Figure4}
# create function to retrieve stats on eQTLs
retrieveeQTLstats <- function(path, tissue, condition){
  data <- read.table(paste0(path, "TMMnormalized_log2CPM_", tissue, "_", condition, "_pvalcorrected.txt"), stringsAsFactors=FALSE)
  # retrieve number of significant eQTLs
  sig <- length(which(data$adjustedpvalue<0.05))
  # retrieve total number of expressed genes
  tot <- nrow(data)
  
  datafull <- read.table(paste0(path, "TMMnormalized_log2CPM_", tissue, "_", condition, ".txt"), stringsAsFactors=FALSE)
  neg <- table(sign(datafull$V9[which(data$adjustedpvalue<0.05)]))[["-1"]]
  pos <- table(sign(datafull$V9[which(data$adjustedpvalue<0.05)]))[["1"]]
  
  # calculate percentage of significant eQTLs (over number of expressed genes)
  percentage_sig <- sig/tot*100
  # calculate percentage of significant eQTLs with positive slope (over number of expressed genes)
  percentage_pos <- pos/tot*100
  # calculate percentage of difference between significant eQTLs with negative and positive slope (over number of expressed genes)
  percentage_skewness <- (neg-pos)/tot*100
  
  return(c(sig=sig, total=tot, neg=neg, pos=pos, percentage_sig=percentage_sig, percentage_pos=percentage_pos, percentage_skewness=percentage_skewness))
}

# define paths to find data
paths <- c(B6="F:/BXD/data/transcriptome/eQTL_B6mm10primaryassembly_withannotation/", BXD="F:/BXD/data/transcriptome/eQTL_BXDfromGNandimputedD2blocks_withannotation/")

# load data
CNSDcomparisonstats <- t(sapply(paths, FUN=retrieveeQTLstats, tissue="Cortex", condition="NSD"))
CSDcomparisonstats <- t(sapply(paths, FUN=retrieveeQTLstats, tissue="Cortex", condition="SD"))
LNSDcomparisonstats <- t(sapply(paths, FUN=retrieveeQTLstats, tissue="Liver", condition="NSD"))
LSDcomparisonstats <- t(sapply(paths, FUN=retrieveeQTLstats, tissue="Liver", condition="SD"))

# group data
comparisonSig <- cbind(CNSDcomparisonstats[, "percentage_sig"], CSDcomparisonstats[, "percentage_sig"], LNSDcomparisonstats[, "percentage_sig"], LSDcomparisonstats[, "percentage_sig"])
comparisonSkewness <- cbind(CNSDcomparisonstats[, "percentage_skewness"], CSDcomparisonstats[, "percentage_skewness"], LNSDcomparisonstats[, "percentage_skewness"], LSDcomparisonstats[, "percentage_skewness"])

# function to check if difference is within 5% of average
percentagediff <- function(x1, x2){
  diff <- x1-x2
  average <- apply(cbind(x1, x2), MARGIN=1, FUN=mean)

  return(abs(diff/average))
}

# load, process, and plot local eQTL comparison
plotVennSimilarat5percent <- function(tissue, condition){
  # Loading data
  eQTLB6 <- read.table(paste0("F:/BXD/data/transcriptome/eQTL_B6mm10primaryassembly_withannotation/TMMnormalized_log2CPM_", tissue, "_", condition, "_grouped.tab"), header=TRUE, row.names=1, stringsAsFactors=FALSE)
  eQTLBXD <- read.table(paste0("F:/BXD/data/transcriptome/eQTL_BXDfromGNandimputedD2blocks_withannotation/TMMnormalized_log2CPM_", tissue, "_", condition, "_grouped.tab"), header=TRUE, row.names=1, stringsAsFactors=FALSE)

  # remove genes with marker NA
  ce_mm10_noNA <- eQTLB6[which(!is.na(eQTLB6$marker)), ]
  ce_BXD_noNA <- eQTLBXD[which(!is.na(eQTLBXD$marker)), ]

  commongenes <- intersect(rownames(ce_mm10_noNA), rownames(ce_BXD_noNA))
  samemarker <- commongenes[which(ce_BXD_noNA[commongenes, "marker"]==ce_mm10_noNA[commongenes, "marker"])]
  similarslope <- commongenes[which(percentagediff(ce_BXD_noNA[commongenes, "slope"], ce_mm10_noNA[commongenes, "slope"])<0.05)]
  similarqvalue <- commongenes[which(percentagediff(ce_BXD_noNA[commongenes, "adjustedpvalue"], ce_mm10_noNA[commongenes, "adjustedpvalue"])<0.05)]

  venn.plot <- venn.diagram(
    x=list(marker=samemarker, slope=similarslope, qvalue=similarqvalue),
    filename=NULL,
    main=paste0("local eQTL in ", tissue, " ", condition, " (similar at 5%)"),
    fontfamily="sans",
    main.fontfamily="sans",
    cex=0.8,
    cat.cex=0.8,
    main.cex=0.9,
    cat.fontfamily="sans",
    sigdigs=2,
    print.mode=c("raw", "percent"),
    fill=c("lightgreen", "cornflowerblue", "red"),
    alpha=c(0.75, 0.5, 0.25)
  );

  ###grid.draw(venn.plot)
}
```

```{r Figure4_ConsequencesOfMappingReferenceAtlocaleQTLlevel, fig.width=7, fig.height=5}
par(mfrow=c(2,2), bty="l", col.lab="grey30", col.axis="grey30", col.main="grey30", fg="grey30")
grid.newpage()
# panel A
barplot(comparisonSig, names.arg=rep(c("GRCm38", "BXD"),4), beside=TRUE, las=2, ylab="significant eQTLs %", ylim=c(0, 35))
text(x=c(1:4*2+1:4-1), y=37, labels=c("Cortex NSD", "Cortex SD", "Liver NSD", "Liver SD"), cex=0.8, xpd=TRUE)
abline(h=0)

# panel B
barplot(comparisonSkewness, names.arg=rep(c("GRCm38", "BXD"),4), beside=TRUE, las=2, ylab="Skewness %", ylim=c(-0.5, 2.5))
text(x=c(1:4*2+1:4-1), y=2.7, labels=c("Cortex NSD", "Cortex SD", "Liver NSD", "Liver SD"), cex=0.8, xpd=TRUE)
abline(h=0)

# panel C
fig4c <- plotVennSimilarat5percent(tissue="Cortex", condition="NSD")

# panel D
fig4d <- plotVennSimilarat5percent(tissue="Liver", condition="NSD")

frame()
vps <- baseViewports()
pushViewport(vps$inner, vps$figure, vps$plot)
try(grid.draw(), silent=TRUE)
popViewport(0)
pushViewport(viewport(layout = grid.layout(2, 2, widths=unit(c(0.5, 0.5), "npc"))))
pushViewport(viewport(layout.pos.col=1, layout.pos.row=2))
grid.draw(fig4c)
popViewport(0)
pushViewport(viewport(layout = grid.layout(2, 2, widths=unit(c(0.5, 0.5), "npc"))))
pushViewport(viewport(layout.pos.col=2, layout.pos.row=2))
grid.draw(fig4d)
popViewport(0)

# add panel labels
mtext(LETTERS[1:3], side=3, line=-3, adj=c(0,0.52,0), padj=c(0,0,11.5), cex=1.5, outer=TRUE)
```

__Figure 4 | Consequences of mapping reference at local eQTL level.__

A) Percentage of significant (FDR 5%) local eQTLs with GRCm38 or BXD-specific references.
B) Percentage of skewness of significant(FDR 5%) local eQTLs slope with GRCm38 or BXD-specific references.
C) Local eQTLs overlapping between GRCm38 and BXD-specific references in cortex NSD (left) or in liver NSD (right). The sign of the slope measures the direction of gene expression (B6 or D2 as more expressed allele).


## Figure 5: Evaluating mapping parameters.

```{r prepare_Figure5}
basepath <- "F:/BXD/data/MappingEvaluation/"
settinglist <- list.dirs(path=basepath, full.names=FALSE, recursive=FALSE)
tissuelist <- c("Cortex", "Liver")
conditionlist <- c("NSD", "SD")
normalizationtypelist <- c("log2CPM")

# function to retrieve number of eQTLs and number of expressed genes
retrieveeQTLstats <- function(basepath, setting, normalizationtype, tissue, condition){
  # load eQTL data
  ##print(paste0(basepath, setting, "/TMMnormalized_", normalizationtype, "_", tissue, "_", condition, "_pvalcorrected.txt"))
  data <- read.table(paste0(basepath, setting, "/TMMnormalized_", normalizationtype, "_", tissue, "_", condition, "_pvalcorrected.txt"), stringsAsFactors=FALSE)
  # retrieve number of significant eQTLs
  sig <- length(which(data$adjustedpvalue<0.05))
  # retrieve total number of eQTLs
  tot <- nrow(data)
  
  return(c(nb_sig=sig, total=tot, ratio=sig/tot))
}

# retrieve all values for log2CPM data
normalizationtype <- "log2CPM"
for(tissue in tissuelist){
  for(condition in conditionlist){
      assign(paste("eQTLstats", tissue, condition, sep="_"), as.data.frame(t(sapply(settinglist, retrieveeQTLstats, basepath=basepath, normalizationtype=normalizationtype, tissue=tissue, condition=condition))))
  }
}

# ranking parameters by tissue
averageratio_Cortex <- apply(cbind(eQTLstats_Cortex_NSD$ratio, eQTLstats_Cortex_SD$ratio), MARGIN=1, FUN=mean)
order_Cortex <- sort(averageratio_Cortex, index.return=TRUE)$ix

averageratio_Liver <- apply(cbind(eQTLstats_Liver_NSD$ratio, eQTLstats_Liver_SD$ratio), MARGIN=1, FUN=mean)
order_Liver <- sort(averageratio_Liver, index.return=TRUE)$ix

# function to split and format labels
splitlabels <- function(mylabels){
  ##splitedlabels <- matrix(unlist(strsplit(mylabels, "_")), ncol=4, nrow=40, byrow=TRUE)
  splitedlabels <- matrix(unlist(strsplit(mylabels, "_")), ncol=4, nrow=length(mylabels), byrow=TRUE)
  colnames(splitedlabels) <- c("Annotation", "Trimming", "Splicing", "Mismatches")
  rownames(splitedlabels) <- NULL
  splitedlabels[splitedlabels[,"Annotation"]=="withannotation","Annotation"] <- "black"
  splitedlabels[splitedlabels[,"Annotation"]=="withoutannotation","Annotation"] <- "white"
  splitedlabels[splitedlabels[,"Trimming"]=="Local","Trimming"] <- "black"
  splitedlabels[splitedlabels[,"Trimming"]=="EndToEnd","Trimming"] <- "white"
  splitedlabels[splitedlabels[,"Splicing"]=="1","Splicing"] <- "white"
  splitedlabels[splitedlabels[,"Splicing"]=="0","Splicing"] <- "black"
  return(splitedlabels)
}

# prepare splited labels for each tissue
splitedlabels_Cortex <- splitlabels(na.omit(gsub("genotypesandimputed_", "", rownames(eQTLstats_Cortex_NSD)[order_Cortex])))
splitedlabels_Liver <- splitlabels(na.omit(gsub("genotypesandimputed_", "", rownames(eQTLstats_Liver_NSD)[order_Liver])))
```

```{r Figure5_EvaluatingMappingParameters, fig.width=8, fig.height=7}
# plot all tissues and conditions with merged setting order
# plot all tissues and conditions with merged setting order
layout(matrix(c(1,2,2,2,3,4,4,4), nrow=1, ncol=8, byrow=TRUE))
par(mar=c(5.1, 2.1, 6.1, 2.1))
from <- 1
to <- 19

# legend for Cortex
plot.new()
x <- c(0:3/3)+0.1
y <- seq(0, 1, length.out=nrow(splitedlabels_Cortex))
y4 <- seq(0.025, 1.025, length.out=nrow(splitedlabels_Cortex))
text(x=x-0.15, y=1.04, cex=1.2, labels=colnames(splitedlabels_Cortex), srt=90, xpd=TRUE, pos=4)
text(x=rep(x[4], nrow(splitedlabels_Cortex)), y=y4, pos=1, splitedlabels_Cortex[, "Mismatches"], cex=1.2, xpd=TRUE)
points(x=rep(x[1], nrow(splitedlabels_Cortex)), y=y, pch=22, bg=splitedlabels_Cortex[, "Annotation"], col="black", cex=2)
points(x=rep(x[2], nrow(splitedlabels_Cortex)), y=y, pch=22, bg=splitedlabels_Cortex[, "Trimming"], col="black", cex=2)
points(x=rep(x[3], nrow(splitedlabels_Cortex)), y=y, pch=22, bg=splitedlabels_Cortex[, "Splicing"], col="black", cex=2)

# plot for Cortex
gap.plot(c(0, 30), c(1, 40), gap=c(from,to), gap.axis="x", type="p", pch=19, main=paste("Cortex", normalizationtype), xlab="significant eQTLs %", ylab="", xlim=c(0,30), yticlab=FALSE, ytics=-1, lty=0, col="white", cex.main=1.5, cex.lab=1.4)
abline(h=1:40, lty=3, col="grey70")
gap.plot(averageratio_Cortex[order_Cortex]*100, seq(1, length(averageratio_Cortex[order_Cortex]*100)), gap=c(from,to), gap.axis="x", type="p", pch=19, xlab="significant eQTLs (%)", ylab="", xlim=c(0,30), yticlab=FALSE, ytics=-1, add=TRUE)
axis.break(1, from, breakcol="grey90", style="gap")
axis.break(1, from*(1+0.02), breakcol="black", style="slash")
axis.break(3, from*(1+0.02), breakcol="black", style="slash")

# legend for Liver
plot.new()
x <- c(0:3/3)+0.1
y <- seq(0, 1, length.out=nrow(splitedlabels_Liver))
y4 <- seq(0.025, 1.025, length.out=nrow(splitedlabels_Liver))
text(x=x-0.15, y=1.04, cex=1.2, labels=colnames(splitedlabels_Liver), srt=90, xpd=TRUE, pos=4)
text(x=rep(x[4], nrow(splitedlabels_Liver)), y=y4, pos=1, splitedlabels_Liver[, "Mismatches"], cex=1.2, xpd=TRUE)
points(x=rep(x[1], nrow(splitedlabels_Liver)), y=y, pch=22, bg=splitedlabels_Liver[, "Annotation"], col="black", cex=2)
points(x=rep(x[2], nrow(splitedlabels_Liver)), y=y, pch=22, bg=splitedlabels_Liver[, "Trimming"], col="black", cex=2)
points(x=rep(x[3], nrow(splitedlabels_Liver)), y=y, pch=22, bg=splitedlabels_Liver[, "Splicing"], col="black", cex=2)

# plot for Liver
gap.plot(c(0, 30), c(1, 40), gap=c(from,to), gap.axis="x", type="p", pch=19, main=paste("Liver", normalizationtype), xlab="significant eQTLs %", ylab="", xlim=c(0,30), yticlab=FALSE, ytics=-1, lty=0, cex.main=1.5, cex.lab=1.4, col="white")
abline(h=1:40, lty=3, col="grey70")
gap.plot(averageratio_Liver[order_Liver]*100, seq(1, length(averageratio_Liver[order_Liver]*100)), gap=c(from,to), gap.axis="x", type="p", pch=19, xlab="significant eQTLs (%)", ylab="", xlim=c(0,30), yticlab=FALSE, ytics=-1, add=TRUE)
axis.break(1, from, breakcol="grey90", style="gap")
axis.break(1, from*(1+0.02), breakcol="black", style="slash")
axis.break(3, from*(1+0.02), breakcol="black", style="slash")
mtext(LETTERS[1:2], side=3, line=0, adj=c(0,0.51), padj=c(1.5,1.5), cex=1.5, outer=TRUE)
```

__Figure 5 | Evaluating mapping parameters.__

A) The performance on local eQTLs of selected mapping settings on cortex samples (average between NSD and SD conditions) is measured by the percentage of expressed genes than have a significant local eQTLs.
B) The performance on local eQTLs of selected mapping settings on liver samples (average between NSD and SD conditions) is measured by the percentage of expressed genes than have a significant local eQTLs.


# R session info

```{r sessionInfo}
sessionInfo()
```
