---
title: "Mapping RNA on genome"
author: "Nastassia Gobet"
date: "4 September 2019"
output:
  html_document:
    code_folding: hide
    fig_caption: yes
    highlight: tango
    number_sections: no
    theme: readable
    toc: yes
    toc_depth: 4
    df_print: paged
  pdf_document:
    fig_caption: yes
  word_document:
    toc: yes
    toc_depth: '4'
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo=TRUE, fig.path="Graphs/", warning=TRUE)

#load needed libraries
library(knitr)

# set working directory
##setwd("F:/BXD/analyses/Comparison_mapping_B6vsD2/OnGenome")
```


# Data

Loading data

```{r}
# load data

# test parameters
Testpar <- read.table("F:/BXD/data/transcriptome/2_mapping_STAR/OnGenome/stats_test/MappingStatisticsTEST_LB61nsd_onD2.tsv", header=FALSE, stringsAsFactors=FALSE)

# load stats for all samples
B6mm10all <- read.table("F:/BXD/data/transcriptome/2_mapping_STAR/OnGenome/MappingStatisticsB6mm10_primaryassembly.tsv", header=TRUE, stringsAsFactors=FALSE, row.names="Sample_Name")
D2all <- read.table("F:/BXD/data/transcriptome/2_mapping_STAR/OnGenome/MappingStatisticsD2.tsv", header=TRUE, stringsAsFactors=FALSE, row.names="Sample_Name")

# load table to convert mouse lines names
MouseLines <- read.table("F:/BXD/data/ConvertLineNames.tsv", sep="\t", header=FALSE, stringsAsFactors=FALSE)
```

B6 mm10 is B6 mm10 primary assembly (no patches).

# Testing parameters

Just an idea about how the different parameters individually affect the uniquely mapped percentage.
TESTfree is the default parameters of STAR.

```{r}
# plot impact of different parameters tested individually.
par(mar=c(8.1,4.1,2.1,2.1))
xcor <- barplot(Testpar$V2[5:14], axisnames=FALSE, las=2, ylab="Uniquely mapped reads %", ylim=c(0,85), col=c("white","grey90","grey80", "grey70","black", "darkred","blue", "darkblue", "skyblue", "forestgreen"))
# add values above bars
text(x=xcor, y=Testpar$V2[5:14], label=Testpar$V2[5:14], pos=3, cex=0.9, col="grey")
#xlab
text(xcor, par("usr")[3], labels=Testpar$V1[5:14], srt=50, adj=c(1.1,1.1), xpd=TRUE, cex=0.9)

# estimate uniquely mapped percentage if cumulative restriction (additive)
sum(Testpar$V2[9]-Testpar$V2[c(5,10,11,14)])
```


# Unique percentage values {.tabset}

## on B6 mm10

```{r}
# display values B6 mm10
B6mm10all
```

## on D2

```{r}
# display values D2
D2all
```

# Analysis

## Parental samples

0 mismatch allowed

```{r}
# get parental samples indexes
CB6samples_idx <- intersect(grep("B6[12]", rownames(D2all)), grep("L", rownames(D2all), invert=TRUE))
CDBsamples_idx <- intersect(grep("DB[12]", rownames(D2all)), grep("L", rownames(D2all), invert=TRUE))
LB6samples_idx <- intersect(grep("B6[12]", rownames(D2all)), grep("L", rownames(D2all)))
LDBsamples_idx <- intersect(grep("DB[12]", rownames(D2all)), grep("L", rownames(D2all)))
# get parental and F1
parentalF1samples_idx <- rownames(D2all)[grep("[DB]", rownames(D2all))]

# Difference in mappability by reference for uniquely exactly mapped reads
par(mfrow=c(1,2))

# in cortex
plot("", las=2, xlim=c(1,2), ylim=c(46,59), bty="l", xaxt="n", main="Cortex", xlab="Reference", ylab="Uniquely exactly mapped reads (%)")
axis(1, at=1:2, labels=c("B6mm10","D2"))
segments(x0=rep(2,8), y0=D2all[c(CB6samples_idx,CDBsamples_idx),"Uniq0"], x1=rep(1,8), y1=B6mm10all[c(CB6samples_idx,CDBsamples_idx),"Uniq0"],
         col=c(rep("grey90",4),rep("wheat",4)), lty = par("lty"), lwd = par("lwd"))
points(rep(1,8),B6mm10all[c(CB6samples_idx,CDBsamples_idx),"Uniq0"], pch=19, cex=1, col=c(rep("grey20",4),rep("tan",4)))
points(rep(2,8),D2all[c(CB6samples_idx,CDBsamples_idx),"Uniq0"], pch=19, cex=1, col=c(rep("grey20",4),rep("tan",4)))
legend("bottom", title="Samples", legend=c("B6","D2"), pch=19, col=c("grey20","tan"), horiz=TRUE)

# in liver
plot("", las=2, xlim=c(1,2), ylim=c(27,41), bty="l", xaxt="n", main="Liver", xlab="Reference", ylab="Uniquely exactly mapped reads (%)")
axis(1, at=1:2, labels=c("B6mm10","D2"))
segments(x0=rep(2,8), y0=D2all[c(LB6samples_idx,LDBsamples_idx),"Uniq0"], x1=rep(1,8), y1=B6mm10all[c(LB6samples_idx,LDBsamples_idx),"Uniq0"],
         col=c(rep("grey90",4),rep("wheat",4)), lty=par("lty"), lwd=1)
points(rep(1,8),B6mm10all[c(LB6samples_idx,LDBsamples_idx),"Uniq0"], pch=19, cex=1, col=c(rep("grey20",4),rep("tan",4)))
points(rep(2,8),D2all[c(LB6samples_idx,LDBsamples_idx),"Uniq0"], pch=19, cex=1, col=c(rep("grey20",4),rep("tan",4)))
legend("bottom", title="Samples", legend=c("B6","D2"), pch=19, col=c("grey20","tan"), horiz=TRUE)
```

Observations:

* In cortex, B6 samples map better on B6 reference, and D2 samples map better on D2.
* In liver, B6 samples still map better on B6 reference than on D2 reference, but D2 samples seem to map more similarly to both genomes.


## Parental and F1 samples {.tabset}

### 0

```{r}
# exactly matching
plot(B6mm10all[parentalF1samples_idx,"Uniq0"], pch=19, col="black", ylim=c(29,77), las=2, xaxt="n", ylab="Uniquely mapped percentage", xlab="Samples", main="0 mismatch allowed")
points(D2all[parentalF1samples_idx,"Uniq0"], pch=19, col="tan")
abline(v=c(4.5,6.5,10.5,12.5,16.5,18.5,22.5))
axis(1, at=c(4.5,6.5,10.5,12.5,16.5,18.5,22.5), labels=rep(c(""),7))
axis(1, at=c(c(2.5,5.5,8.5,11.5),12+c(2.5,5.5,8.5,11.5)), tick=FALSE, labels=rep(c("B6","BxD","D2","DxB"),2))
for(myline in seq(0.5,2,0.5)){
  axis(1, at=12.5, line=myline, labels=c(""))
}
axis(1, at=c(6.5,18.5), line=1, tick=FALSE, labels=c("cortex", "liver"))
legend("bottomleft", title="Reference", legend=c("B6mm10","D2"), fill=c("black","tan"))
```

Observations:

* About 15% of difference in mappability between cortex and liver (liver lower). The variability seems globally similar.
* B6 and D2 samples behave as expected (B6 samples maps higher on B6 reference, D2 samples maps higher on D2 reference), though the difference is very low for liver D2 samples mapped on D2.
* F1 samples map higher on B6 than on D2.


### 1

```{r}
# 1 mismatch allowed
plot(B6mm10all[parentalF1samples_idx,"Uniq1"], pch=19, col="black", ylim=c(29,77), las=2, xaxt="n", ylab="Uniquely mapped percentage", xlab="Samples", main="1 mismatch allowed")
points(D2all[parentalF1samples_idx,"Uniq1"], pch=19, col="tan")
abline(v=c(4.5,6.5,10.5,12.5,16.5,18.5,22.5))
axis(1, at=c(4.5,6.5,10.5,12.5,16.5,18.5,22.5), labels=rep(c(""),7))
axis(1, at=c(c(2.5,5.5,8.5,11.5),12+c(2.5,5.5,8.5,11.5)), tick=FALSE, labels=rep(c("B6","BxD","D2","DxB"),2))
for(myline in seq(0.5,2,0.5)){
  axis(1, at=12.5, line=myline, labels=c(""))
}
axis(1, at=c(6.5,18.5), line=1, tick=FALSE, labels=c("cortex", "liver"))
legend("bottomleft", title="Reference", legend=c("B6mm10","D2"), fill=c("black","tan"))

```


### 2

```{r}
# 2 mismatches allowed
plot(B6mm10all[parentalF1samples_idx,"Uniq2"], pch=19, col="black", ylim=c(29,77), las=2, xaxt="n", ylab="Uniquely mapped percentage", xlab="Samples", main="2 mismatches allowed")
points(D2all[parentalF1samples_idx,"Uniq2"], pch=19, col="tan")
abline(v=c(4.5,6.5,10.5,12.5,16.5,18.5,22.5))
axis(1, at=c(4.5,6.5,10.5,12.5,16.5,18.5,22.5), labels=rep(c(""),7))
axis(1, at=c(c(2.5,5.5,8.5,11.5),12+c(2.5,5.5,8.5,11.5)), tick=FALSE, labels=rep(c("B6","BxD","D2","DxB"),2))
for(myline in seq(0.5,2,0.5)){
  axis(1, at=12.5, line=myline, labels=c(""))
}
axis(1, at=c(6.5,18.5), line=1, tick=FALSE, labels=c("cortex", "liver"))
legend("bottomleft", title="Reference", legend=c("B6mm10","D2"), fill=c("black","tan"))

```

### 3

```{r}
# 3 mismatches allowed
plot(B6mm10all[parentalF1samples_idx,"Uniq3"], pch=19, col="black", ylim=c(29,77), las=2, xaxt="n", ylab="Uniquely mapped percentage", xlab="Samples", main="3 mismatches allowed")
points(D2all[parentalF1samples_idx,"Uniq3"], pch=19, col="tan")
abline(v=c(4.5,6.5,10.5,12.5,16.5,18.5,22.5))
axis(1, at=c(4.5,6.5,10.5,12.5,16.5,18.5,22.5), labels=rep(c(""),7))
axis(1, at=c(c(2.5,5.5,8.5,11.5),12+c(2.5,5.5,8.5,11.5)), tick=FALSE, labels=rep(c("B6","BxD","D2","DxB"),2))
for(myline in seq(0.5,2,0.5)){
  axis(1, at=12.5, line=myline, labels=c(""))
}
axis(1, at=c(6.5,18.5), line=1, tick=FALSE, labels=c("cortex", "liver"))
legend("bottomleft", title="Reference", legend=c("B6mm10","D2"), fill=c("black","tan"))

```

With 2 or 3 mismatches allowed, the F1 samples seem to map more similarly to both references.

### 4

```{r}
# 4 mismatches allowed
plot(B6mm10all[parentalF1samples_idx,"Uniq4"], pch=19, col="black", ylim=c(29,77), las=2, xaxt="n", ylab="Uniquely mapped percentage", xlab="Samples", main="4 mismatches allowed")
points(D2all[parentalF1samples_idx,"Uniq4"], pch=19, col="tan")
abline(v=c(4.5,6.5,10.5,12.5,16.5,18.5,22.5))
axis(1, at=c(4.5,6.5,10.5,12.5,16.5,18.5,22.5), labels=rep(c(""),7))
axis(1, at=c(c(2.5,5.5,8.5,11.5),12+c(2.5,5.5,8.5,11.5)), tick=FALSE, labels=rep(c("B6","BxD","D2","DxB"),2))
for(myline in seq(0.5,2,0.5)){
  axis(1, at=12.5, line=myline, labels=c(""))
}
axis(1, at=c(6.5,18.5), line=1, tick=FALSE, labels=c("cortex", "liver"))
legend("bottomleft", title="Reference", legend=c("B6mm10","D2"), fill=c("black","tan"))

```

### 5

```{r}
# 5 mismatches allowed
plot(B6mm10all[parentalF1samples_idx,"Uniq5"], pch=19, col="black", ylim=c(29,77), las=2, xaxt="n", ylab="Uniquely mapped percentage", xlab="Samples", main="5 mismatches allowed")
points(D2all[parentalF1samples_idx,"Uniq5"], pch=19, col="tan")
abline(v=c(4.5,6.5,10.5,12.5,16.5,18.5,22.5))
axis(1, at=c(4.5,6.5,10.5,12.5,16.5,18.5,22.5), labels=rep(c(""),7))
axis(1, at=c(c(2.5,5.5,8.5,11.5),12+c(2.5,5.5,8.5,11.5)), tick=FALSE, labels=rep(c("B6","BxD","D2","DxB"),2))
for(myline in seq(0.5,2,0.5)){
  axis(1, at=12.5, line=myline, labels=c(""))
}
axis(1, at=c(6.5,18.5), line=1, tick=FALSE, labels=c("cortex", "liver"))
legend("bottomleft", title="Reference", legend=c("B6mm10","D2"), fill=c("black","tan"))
```

### 10

```{r}
# 10 mismatches allowed
plot(B6mm10all[parentalF1samples_idx,"Uniq10"], pch=19, col="black", ylim=c(29,77), las=2, xaxt="n", ylab="Uniquely mapped percentage", xlab="Samples", main="10 mismatches allowed")
points(D2all[parentalF1samples_idx,"Uniq10"], pch=19, col="tan")
abline(v=c(4.5,6.5,10.5,12.5,16.5,18.5,22.5))
axis(1, at=c(4.5,6.5,10.5,12.5,16.5,18.5,22.5), labels=rep(c(""),7))
axis(1, at=c(c(2.5,5.5,8.5,11.5),12+c(2.5,5.5,8.5,11.5)), tick=FALSE, labels=rep(c("B6","BxD","D2","DxB"),2))
for(myline in seq(0.5,2,0.5)){
  axis(1, at=12.5, line=myline, labels=c(""))
}
axis(1, at=c(6.5,18.5), line=1, tick=FALSE, labels=c("cortex", "liver"))
legend("bottomleft", title="Reference", legend=c("B6mm10","D2"), fill=c("black","tan"))
```

At 10 mismatches (maximum allowed by STAR default parameters), bias for D2 reference (similar to what observed previously on transcriptome).

## All samples {.tabset}

### 0

```{r}
# exactly matching
plot(B6mm10all$Uniq0, pch=19, col="black", ylim=c(29,77), las=2, xaxt="n", ylab="Uniquely mapped percentage", xlab="Samples", main="0 mismatch allowed")
points(D2all$Uniq0, pch=19, col="tan")
##abline(v=c(4.5,6.5,10.5,12.5,16.5,18.5,22.5))
##axis(1, at=c(4.5,6.5,10.5,12.5,16.5,18.5,22.5), labels=rep(c(""),7))
##axis(1, at=c(c(2.5,5.5,8.5,11.5),12+c(2.5,5.5,8.5,11.5)), tick=FALSE, labels=rep(c("B6","BxD","D2","DxB"),2))
for(myline in seq(0,2,0.5)){
  axis(1, at=88, line=myline, labels=c(""))
}
axis(1, at=c(44.5,132.5), line=1, tick=FALSE, labels=c("cortex", "liver"))
legend("bottomleft", title="Reference", legend=c("B6mm10","D2"), fill=c("black","tan"))
```


```{r}
# Difference in mappability by reference for uniquely exactly mapped reads
par(mfrow=c(1,2))

# in cortex
plot("", las=2, xlim=c(1,2), ylim=c(45,57), bty="l", xaxt="n", main="Cortex", xlab="Reference", ylab="Uniquely exactly mapped reads (%)")
axis(1, at=1:2, labels=c("B6mm10","D2"))
segments(x0=rep(2,176), y0=D2all[,"Uniq0"], x1=rep(1,176), y1=B6mm10all[,"Uniq0"],
         lty = par("lty"), lwd = par("lwd"))
points(rep(1,176),B6mm10all[,"Uniq0"], pch=19, cex=1)
points(rep(2,176),D2all[,"Uniq0"], pch=19, cex=1)

# in liver
plot("", las=2, xlim=c(1,2), ylim=c(27,41), bty="l", xaxt="n", main="Liver", xlab="Reference", ylab="Uniquely exactly mapped reads (%)")
axis(1, at=1:2, labels=c("B6mm10","D2"))
segments(x0=rep(2,176), y0=D2all[,"Uniq0"], x1=rep(1,176), y1=B6mm10all[,"Uniq0"],
         lty=par("lty"), lwd=1)
points(rep(1,176),B6mm10all[,"Uniq0"], pch=19, cex=1)
points(rep(2,176),D2all[,"Uniq0"], pch=19, cex=1)
```

### 1

```{r}
# 1 mismatch allowed
plot(B6mm10all$Uniq1, pch=19, col="black", ylim=c(29,77), las=2, xaxt="n", ylab="Uniquely mapped percentage", xlab="Samples", main="1 mismatch allowed")
points(D2all$Uniq1, pch=19, col="tan")
for(myline in seq(0,2,0.5)){
  axis(1, at=88, line=myline, labels=c(""))
}
axis(1, at=c(44.5,132.5), line=1, tick=FALSE, labels=c("cortex", "liver"))
legend("bottomleft", title="Reference", legend=c("B6mm10","D2"), fill=c("black","tan"))
```

### 2

```{r}
# 2 mismatches allowed
plot(B6mm10all$Uniq2, pch=19, col="black", ylim=c(29,77), las=2, xaxt="n", ylab="Uniquely mapped percentage", xlab="Samples", main="2 mismatches allowed")
points(D2all$Uniq2, pch=19, col="tan")
for(myline in seq(0,2,0.5)){
  axis(1, at=88, line=myline, labels=c(""))
}
axis(1, at=c(44.5,132.5), line=1, tick=FALSE, labels=c("cortex", "liver"))
legend("bottomleft", title="Reference", legend=c("B6mm10","D2"), fill=c("black","tan"))
```

### 3

```{r}
# 3 mismatches allowed
plot(B6mm10all$Uniq3, pch=19, col="black", ylim=c(29,77), las=2, xaxt="n", ylab="Uniquely mapped percentage", xlab="Samples", main="3 mismatches allowed")
points(D2all$Uniq3, pch=19, col="tan")
for(myline in seq(0,2,0.5)){
  axis(1, at=88, line=myline, labels=c(""))
}
axis(1, at=c(44.5,132.5), line=1, tick=FALSE, labels=c("cortex", "liver"))
legend("bottomleft", title="Reference", legend=c("B6mm10","D2"), fill=c("black","tan"))
```

### 4

```{r}
# 4 mismatches allowed
plot(B6mm10all$Uniq4, pch=19, col="black", ylim=c(29,77), las=2, xaxt="n", ylab="Uniquely mapped percentage", xlab="Samples", main="4 mismatches allowed")
points(D2all$Uniq4, pch=19, col="tan")
for(myline in seq(0,2,0.5)){
  axis(1, at=88, line=myline, labels=c(""))
}
axis(1, at=c(44.5,132.5), line=1, tick=FALSE, labels=c("cortex", "liver"))
legend("bottomleft", title="Reference", legend=c("B6mm10","D2"), fill=c("black","tan"))
```

### 5

```{r}
# 5 mismatches allowed
plot(B6mm10all$Uniq5, pch=19, col="black", ylim=c(29,77), las=2, xaxt="n", ylab="Uniquely mapped percentage", xlab="Samples", main="5 mismatches allowed")
points(D2all$Uniq5, pch=19, col="tan")
for(myline in seq(0,2,0.5)){
  axis(1, at=88, line=myline, labels=c(""))
}
axis(1, at=c(44.5,132.5), line=1, tick=FALSE, labels=c("cortex", "liver"))
legend("bottomleft", title="Reference", legend=c("B6mm10","D2"), fill=c("black","tan"))
```

### 10

```{r}
# 10 mismatches allowed
plot(B6mm10all$Uniq10, pch=19, col="black", ylim=c(29,77), las=2, xaxt="n", ylab="Uniquely mapped percentage", xlab="Samples", main="10 mismatches allowed")
points(D2all$Uniq10, pch=19, col="tan")
for(myline in seq(0,2,0.5)){
  axis(1, at=88, line=myline, labels=c(""))
}
axis(1, at=c(44.5,132.5), line=1, tick=FALSE, labels=c("cortex", "liver"))
legend("bottomleft", title="Reference", legend=c("B6mm10","D2"), fill=c("black","tan"))
```


## By sample 

```{r}
par(mfrow=c(2,5))
for(mysample in rownames(B6mm10all)){
  # on B6 mm10 reference
  plot(c(0:5,10),B6mm10all[mysample,], las=1, bty="l", pch=19, ylim=c(floor(min(B6mm10all[mysample,],D2all[mysample,])),ceiling(max(B6mm10all[mysample,],D2all[mysample,]))), xlab="# mismatch allowed", ylab="Uniquely mapped %", main=mysample)
  # on D2 reference
  points(c(0:5,10),D2all[mysample,], las=1, bty="l", pch=19, col="tan")
}
```

Uniquely mapped percentage always increases with higher number of mismatch allowed.
D2 vs B6? liver vs cortex? Shape of increase???


## Genomic regions

Which genomic regions are of interest? Are there a lot of reads mapping between genes?

Use regions with higher coverage (window size? threshold?) and see if they intersect with gene annotation or not.

# Discussion

The use of more restrictive parameters and mapping on the genome instead of transcriptome seems to show:

* Still difference in mappability for the 2 tissues. Still fewer reads map from liver than from cortex.
* Some specificity of sample to its reference (B6 samples map better on B6 reference) instead of all samples map higher on D2. F1 samples map better on B6.
* This specificity seems to decrease by increasing the number of mismatch allowed.

These are due to:

* map on genome vs transcriptome?
* constraints (no indels, no splicing, no soft-clip, no mismatch)?


Various info:

* Mouse lines heterogeneozity: in the genotypes (mm9) merged, B6 shows 93% of "B" (0% of "D", 7% of "H", <1% of "U"), D2 shows 81% of "D" (0% of "B", 19% of "H", <1% of "U"). (H=heterogenous, B=homozygote B6, D=homozygote D2, U=Unknown). In the genotypes (mm9) from GeneNetwork there is no heterogenous loci for parental strains.
* Using "free" (STAR default) parameters, 77% of uniquely mapped reads on genome compared to 75.78% (unfiltered) on transcriptome (first replicate B6 NSD liver on D2 reference).
* Using restrictive parameters, 46.16% of uniquely mapped reads on genome compared to 60.02% (introns allowed) on transcriptome (first replicate B6 NSD cortex on D2 reference).


Ideas to go further:

* Be less restrictive on introns, indels, and soft-clipping? (aka just restrict mismatches)
* Check in other tissues (BXD samples from Bernard Thorens group used by Maxime for variant calling)?
