#BXD70
#DATE: Sa Dez 5 06:01:39 CET 2020
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_0/70nsd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType EndToEnd --outFilterMismatchNmax 0 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_0/70nsd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_0/70nsd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_0/70nsd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_0/70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_0/70sd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType EndToEnd --outFilterMismatchNmax 0 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_0/70sd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_0/70sd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_0/70sd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_0/70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_0/L70nsd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType EndToEnd --outFilterMismatchNmax 0 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_0/L70nsd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_0/L70nsd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_0/L70nsd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_0/L70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_0/L70sd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType EndToEnd --outFilterMismatchNmax 0 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_0/L70sd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_0/L70sd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_0/L70sd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_0/L70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_1/70nsd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType EndToEnd --outFilterMismatchNmax 1 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_1/70nsd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_1/70nsd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_1/70nsd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_1/70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_1/70sd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType EndToEnd --outFilterMismatchNmax 1 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_1/70sd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_1/70sd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_1/70sd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_1/70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_1/L70nsd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType EndToEnd --outFilterMismatchNmax 1 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_1/L70nsd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_1/L70nsd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_1/L70nsd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_1/L70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_1/L70sd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType EndToEnd --outFilterMismatchNmax 1 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_1/L70sd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_1/L70sd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_1/L70sd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_1/L70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_2/70nsd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType EndToEnd --outFilterMismatchNmax 2 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_2/70nsd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_2/70nsd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_2/70nsd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_2/70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_2/70sd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType EndToEnd --outFilterMismatchNmax 2 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_2/70sd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_2/70sd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_2/70sd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_2/70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_2/L70nsd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType EndToEnd --outFilterMismatchNmax 2 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_2/L70nsd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_2/L70nsd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_2/L70nsd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_2/L70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_2/L70sd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType EndToEnd --outFilterMismatchNmax 2 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_2/L70sd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_2/L70sd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_2/L70sd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_2/L70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_3/70nsd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType EndToEnd --outFilterMismatchNmax 3 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_3/70nsd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_3/70nsd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_3/70nsd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_3/70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_3/70sd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType EndToEnd --outFilterMismatchNmax 3 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_3/70sd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_3/70sd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_3/70sd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_3/70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_3/L70nsd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType EndToEnd --outFilterMismatchNmax 3 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_3/L70nsd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_3/L70nsd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_3/L70nsd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_3/L70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_3/L70sd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType EndToEnd --outFilterMismatchNmax 3 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_3/L70sd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_3/L70sd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_3/L70sd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_3/L70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_10/70nsd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType EndToEnd --outFilterMismatchNmax 10 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_10/70nsd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_10/70nsd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_10/70nsd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_10/70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_10/70sd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType EndToEnd --outFilterMismatchNmax 10 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_10/70sd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_10/70sd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_10/70sd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_10/70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_10/L70nsd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType EndToEnd --outFilterMismatchNmax 10 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_10/L70nsd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_10/L70nsd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_10/L70nsd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_10/L70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_10/L70sd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType EndToEnd --outFilterMismatchNmax 10 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_10/L70sd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_10/L70sd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_10/L70sd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_0\_10/L70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_0/70nsd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_0/70nsd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_0/70nsd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_0/70nsd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_0/70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_0/70sd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_0/70sd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_0/70sd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_0/70sd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_0/70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_0/L70nsd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_0/L70nsd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_0/L70nsd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_0/L70nsd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_0/L70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_0/L70sd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_0/L70sd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_0/L70sd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_0/L70sd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_0/L70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_1/70nsd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 1 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_1/70nsd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_1/70nsd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_1/70nsd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_1/70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_1/70sd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 1 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_1/70sd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_1/70sd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_1/70sd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_1/70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_1/L70nsd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 1 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_1/L70nsd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_1/L70nsd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_1/L70nsd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_1/L70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_1/L70sd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 1 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_1/L70sd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_1/L70sd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_1/L70sd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_1/L70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_2/70nsd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 2 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_2/70nsd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_2/70nsd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_2/70nsd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_2/70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_2/70sd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 2 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_2/70sd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_2/70sd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_2/70sd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_2/70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_2/L70nsd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 2 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_2/L70nsd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_2/L70nsd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_2/L70nsd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_2/L70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_2/L70sd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 2 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_2/L70sd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_2/L70sd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_2/L70sd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_2/L70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_3/70nsd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 3 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_3/70nsd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_3/70nsd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_3/70nsd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_3/70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_3/70sd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 3 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_3/70sd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_3/70sd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_3/70sd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_3/70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_3/L70nsd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 3 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_3/L70nsd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_3/L70nsd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_3/L70nsd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_3/L70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_3/L70sd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 3 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_3/L70sd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_3/L70sd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_3/L70sd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_3/L70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_10/70nsd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 10 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_10/70nsd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_10/70nsd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_10/70nsd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_10/70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_10/70sd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 10 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_10/70sd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_10/70sd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_10/70sd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_10/70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_10/L70nsd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 10 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_10/L70nsd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_10/L70nsd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_10/L70nsd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_10/L70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_10/L70sd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 10 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_10/L70sd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_10/L70sd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_10/L70sd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_EndToEnd\_1\_10/L70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_0/70nsd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType Local --outFilterMismatchNmax 0 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_0/70nsd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_0/70nsd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_0/70nsd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_0/70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_0/70sd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType Local --outFilterMismatchNmax 0 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_0/70sd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_0/70sd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_0/70sd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_0/70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_0/L70nsd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType Local --outFilterMismatchNmax 0 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_0/L70nsd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_0/L70nsd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_0/L70nsd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_0/L70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_0/L70sd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType Local --outFilterMismatchNmax 0 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_0/L70sd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_0/L70sd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_0/L70sd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_0/L70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_1/70nsd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType Local --outFilterMismatchNmax 1 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_1/70nsd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_1/70nsd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_1/70nsd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_1/70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_1/70sd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType Local --outFilterMismatchNmax 1 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_1/70sd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_1/70sd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_1/70sd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_1/70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_1/L70nsd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType Local --outFilterMismatchNmax 1 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_1/L70nsd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_1/L70nsd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_1/L70nsd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_1/L70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_1/L70sd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType Local --outFilterMismatchNmax 1 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_1/L70sd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_1/L70sd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_1/L70sd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_1/L70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_2/70nsd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType Local --outFilterMismatchNmax 2 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_2/70nsd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_2/70nsd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_2/70nsd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_2/70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_2/70sd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType Local --outFilterMismatchNmax 2 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_2/70sd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_2/70sd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_2/70sd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_2/70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_2/L70nsd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType Local --outFilterMismatchNmax 2 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_2/L70nsd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_2/L70nsd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_2/L70nsd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_2/L70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_2/L70sd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType Local --outFilterMismatchNmax 2 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_2/L70sd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_2/L70sd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_2/L70sd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_2/L70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_3/70nsd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType Local --outFilterMismatchNmax 3 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_3/70nsd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_3/70nsd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_3/70nsd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_3/70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_3/70sd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType Local --outFilterMismatchNmax 3 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_3/70sd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_3/70sd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_3/70sd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_3/70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_3/L70nsd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType Local --outFilterMismatchNmax 3 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_3/L70nsd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_3/L70nsd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_3/L70nsd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_3/L70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_3/L70sd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType Local --outFilterMismatchNmax 3 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_3/L70sd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_3/L70sd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_3/L70sd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_3/L70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_10/70nsd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType Local --outFilterMismatchNmax 10 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_10/70nsd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_10/70nsd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_10/70nsd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_10/70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_10/70sd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType Local --outFilterMismatchNmax 10 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_10/70sd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_10/70sd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_10/70sd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_10/70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_10/L70nsd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType Local --outFilterMismatchNmax 10 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_10/L70nsd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_10/L70nsd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_10/L70nsd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_10/L70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_10/L70sd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType Local --outFilterMismatchNmax 10 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_10/L70sd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_10/L70sd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_10/L70sd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_0\_10/L70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_0/70nsd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType Local --outFilterMismatchNmax 0 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_0/70nsd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_0/70nsd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_0/70nsd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_0/70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_0/70sd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType Local --outFilterMismatchNmax 0 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_0/70sd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_0/70sd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_0/70sd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_0/70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_0/L70nsd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType Local --outFilterMismatchNmax 0 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_0/L70nsd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_0/L70nsd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_0/L70nsd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_0/L70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_0/L70sd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType Local --outFilterMismatchNmax 0 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_0/L70sd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_0/L70sd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_0/L70sd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_0/L70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_1/70nsd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType Local --outFilterMismatchNmax 1 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_1/70nsd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_1/70nsd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_1/70nsd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_1/70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_1/70sd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType Local --outFilterMismatchNmax 1 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_1/70sd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_1/70sd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_1/70sd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_1/70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_1/L70nsd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType Local --outFilterMismatchNmax 1 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_1/L70nsd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_1/L70nsd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_1/L70nsd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_1/L70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_1/L70sd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType Local --outFilterMismatchNmax 1 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_1/L70sd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_1/L70sd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_1/L70sd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_1/L70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_2/70nsd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType Local --outFilterMismatchNmax 2 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_2/70nsd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_2/70nsd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_2/70nsd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_2/70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_2/70sd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType Local --outFilterMismatchNmax 2 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_2/70sd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_2/70sd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_2/70sd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_2/70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_2/L70nsd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType Local --outFilterMismatchNmax 2 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_2/L70nsd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_2/L70nsd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_2/L70nsd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_2/L70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_2/L70sd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType Local --outFilterMismatchNmax 2 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_2/L70sd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_2/L70sd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_2/L70sd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_2/L70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_3/70nsd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType Local --outFilterMismatchNmax 3 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_3/70nsd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_3/70nsd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_3/70nsd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_3/70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_3/70sd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType Local --outFilterMismatchNmax 3 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_3/70sd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_3/70sd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_3/70sd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_3/70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_3/L70nsd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType Local --outFilterMismatchNmax 3 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_3/L70nsd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_3/L70nsd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_3/L70nsd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_3/L70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_3/L70sd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType Local --outFilterMismatchNmax 3 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_3/L70sd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_3/L70sd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_3/L70sd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_3/L70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_10/70nsd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType Local --outFilterMismatchNmax 10 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_10/70nsd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_10/70nsd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_10/70nsd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_10/70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_10/70sd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType Local --outFilterMismatchNmax 10 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_10/70sd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_10/70sd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_10/70sd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_10/70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_10/L70nsd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType Local --outFilterMismatchNmax 10 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_10/L70nsd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_10/L70nsd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_10/L70nsd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_10/L70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withoutannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_10/L70sd\_ --outSAMmode Full --outStd SAM --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType Local --outFilterMismatchNmax 10 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_10/L70sd\_align.err | htseq-count -f bam -q -s reverse -t exon -m union - /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/paternal.gtf 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_10/L70sd\_htseqcount.err 1> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_10/L70sd\_ReadsPerGene.out.tab; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withoutannotation\_Local\_1\_10/L70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_0/70nsd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_0/70nsd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_0/70nsd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_0/70nsd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_0/70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_0/70sd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_0/70sd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_0/70sd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_0/70sd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_0/70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_0/L70nsd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_0/L70nsd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_0/L70nsd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_0/L70nsd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_0/L70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_0/L70sd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_0/L70sd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_0/L70sd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_0/L70sd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_0/L70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_1/70nsd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType EndToEnd --outFilterMismatchNmax 1 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_1/70nsd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_1/70nsd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_1/70nsd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_1/70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_1/70sd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType EndToEnd --outFilterMismatchNmax 1 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_1/70sd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_1/70sd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_1/70sd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_1/70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_1/L70nsd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType EndToEnd --outFilterMismatchNmax 1 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_1/L70nsd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_1/L70nsd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_1/L70nsd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_1/L70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_1/L70sd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType EndToEnd --outFilterMismatchNmax 1 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_1/L70sd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_1/L70sd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_1/L70sd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_1/L70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_2/70nsd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType EndToEnd --outFilterMismatchNmax 2 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_2/70nsd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_2/70nsd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_2/70nsd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_2/70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_2/70sd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType EndToEnd --outFilterMismatchNmax 2 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_2/70sd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_2/70sd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_2/70sd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_2/70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_2/L70nsd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType EndToEnd --outFilterMismatchNmax 2 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_2/L70nsd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_2/L70nsd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_2/L70nsd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_2/L70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_2/L70sd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType EndToEnd --outFilterMismatchNmax 2 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_2/L70sd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_2/L70sd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_2/L70sd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_2/L70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_3/70nsd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType EndToEnd --outFilterMismatchNmax 3 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_3/70nsd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_3/70nsd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_3/70nsd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_3/70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_3/70sd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType EndToEnd --outFilterMismatchNmax 3 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_3/70sd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_3/70sd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_3/70sd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_3/70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_3/L70nsd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType EndToEnd --outFilterMismatchNmax 3 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_3/L70nsd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_3/L70nsd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_3/L70nsd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_3/L70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_3/L70sd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType EndToEnd --outFilterMismatchNmax 3 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_3/L70sd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_3/L70sd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_3/L70sd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_3/L70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_10/70nsd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType EndToEnd --outFilterMismatchNmax 10 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_10/70nsd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_10/70nsd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_10/70nsd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_10/70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_10/70sd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType EndToEnd --outFilterMismatchNmax 10 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_10/70sd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_10/70sd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_10/70sd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_10/70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_10/L70nsd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType EndToEnd --outFilterMismatchNmax 10 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_10/L70nsd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_10/L70nsd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_10/L70nsd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_10/L70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_10/L70sd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType EndToEnd --outFilterMismatchNmax 10 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_10/L70sd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_10/L70sd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_10/L70sd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_0\_10/L70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_0/70nsd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_0/70nsd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_0/70nsd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_0/70nsd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_0/70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_0/70sd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_0/70sd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_0/70sd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_0/70sd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_0/70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_0/L70nsd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_0/L70nsd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_0/L70nsd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_0/L70nsd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_0/L70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_0/L70sd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_0/L70sd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_0/L70sd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_0/L70sd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_0/L70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_1/70nsd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 1 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_1/70nsd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_1/70nsd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_1/70nsd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_1/70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_1/70sd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 1 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_1/70sd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_1/70sd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_1/70sd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_1/70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_1/L70nsd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 1 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_1/L70nsd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_1/L70nsd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_1/L70nsd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_1/L70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_1/L70sd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 1 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_1/L70sd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_1/L70sd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_1/L70sd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_1/L70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_2/70nsd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 2 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_2/70nsd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_2/70nsd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_2/70nsd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_2/70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_2/70sd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 2 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_2/70sd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_2/70sd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_2/70sd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_2/70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_2/L70nsd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 2 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_2/L70nsd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_2/L70nsd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_2/L70nsd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_2/L70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_2/L70sd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 2 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_2/L70sd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_2/L70sd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_2/L70sd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_2/L70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_3/70nsd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 3 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_3/70nsd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_3/70nsd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_3/70nsd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_3/70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_3/70sd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 3 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_3/70sd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_3/70sd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_3/70sd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_3/70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_3/L70nsd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 3 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_3/L70nsd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_3/L70nsd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_3/L70nsd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_3/L70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_3/L70sd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 3 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_3/L70sd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_3/L70sd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_3/L70sd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_3/L70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_10/70nsd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 10 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_10/70nsd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_10/70nsd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_10/70nsd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_10/70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_10/70sd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 10 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_10/70sd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_10/70sd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_10/70sd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_10/70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_10/L70nsd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 10 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_10/L70nsd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_10/L70nsd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_10/L70nsd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_10/L70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_10/L70sd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 10 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_10/L70sd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_10/L70sd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_10/L70sd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_EndToEnd\_1\_10/L70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_0/70nsd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType Local --outFilterMismatchNmax 0 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_0/70nsd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_0/70nsd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_0/70nsd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_0/70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_0/70sd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType Local --outFilterMismatchNmax 0 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_0/70sd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_0/70sd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_0/70sd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_0/70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_0/L70nsd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType Local --outFilterMismatchNmax 0 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_0/L70nsd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_0/L70nsd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_0/L70nsd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_0/L70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_0/L70sd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType Local --outFilterMismatchNmax 0 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_0/L70sd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_0/L70sd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_0/L70sd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_0/L70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_1/70nsd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType Local --outFilterMismatchNmax 1 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_1/70nsd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_1/70nsd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_1/70nsd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_1/70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_1/70sd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType Local --outFilterMismatchNmax 1 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_1/70sd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_1/70sd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_1/70sd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_1/70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_1/L70nsd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType Local --outFilterMismatchNmax 1 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_1/L70nsd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_1/L70nsd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_1/L70nsd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_1/L70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_1/L70sd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType Local --outFilterMismatchNmax 1 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_1/L70sd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_1/L70sd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_1/L70sd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_1/L70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_2/70nsd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType Local --outFilterMismatchNmax 2 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_2/70nsd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_2/70nsd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_2/70nsd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_2/70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_2/70sd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType Local --outFilterMismatchNmax 2 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_2/70sd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_2/70sd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_2/70sd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_2/70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_2/L70nsd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType Local --outFilterMismatchNmax 2 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_2/L70nsd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_2/L70nsd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_2/L70nsd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_2/L70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_2/L70sd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType Local --outFilterMismatchNmax 2 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_2/L70sd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_2/L70sd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_2/L70sd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_2/L70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_3/70nsd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType Local --outFilterMismatchNmax 3 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_3/70nsd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_3/70nsd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_3/70nsd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_3/70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_3/70sd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType Local --outFilterMismatchNmax 3 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_3/70sd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_3/70sd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_3/70sd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_3/70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_3/L70nsd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType Local --outFilterMismatchNmax 3 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_3/L70nsd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_3/L70nsd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_3/L70nsd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_3/L70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_3/L70sd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType Local --outFilterMismatchNmax 3 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_3/L70sd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_3/L70sd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_3/L70sd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_3/L70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_10/70nsd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType Local --outFilterMismatchNmax 10 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_10/70nsd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_10/70nsd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_10/70nsd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_10/70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_10/70sd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType Local --outFilterMismatchNmax 10 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_10/70sd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_10/70sd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_10/70sd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_10/70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_10/L70nsd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType Local --outFilterMismatchNmax 10 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_10/L70nsd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_10/L70nsd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_10/L70nsd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_10/L70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_10/L70sd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 0 --alignEndsType Local --outFilterMismatchNmax 10 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_10/L70sd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_10/L70sd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_10/L70sd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_0\_10/L70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_0/70nsd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType Local --outFilterMismatchNmax 0 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_0/70nsd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_0/70nsd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_0/70nsd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_0/70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_0/70sd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType Local --outFilterMismatchNmax 0 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_0/70sd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_0/70sd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_0/70sd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_0/70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_0/L70nsd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType Local --outFilterMismatchNmax 0 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_0/L70nsd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_0/L70nsd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_0/L70nsd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_0/L70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_0/L70sd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType Local --outFilterMismatchNmax 0 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_0/L70sd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_0/L70sd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_0/L70sd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_0/L70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_1/70nsd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType Local --outFilterMismatchNmax 1 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_1/70nsd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_1/70nsd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_1/70nsd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_1/70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_1/70sd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType Local --outFilterMismatchNmax 1 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_1/70sd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_1/70sd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_1/70sd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_1/70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_1/L70nsd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType Local --outFilterMismatchNmax 1 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_1/L70nsd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_1/L70nsd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_1/L70nsd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_1/L70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_1/L70sd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType Local --outFilterMismatchNmax 1 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_1/L70sd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_1/L70sd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_1/L70sd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_1/L70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_2/70nsd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType Local --outFilterMismatchNmax 2 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_2/70nsd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_2/70nsd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_2/70nsd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_2/70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_2/70sd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType Local --outFilterMismatchNmax 2 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_2/70sd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_2/70sd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_2/70sd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_2/70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_2/L70nsd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType Local --outFilterMismatchNmax 2 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_2/L70nsd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_2/L70nsd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_2/L70nsd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_2/L70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_2/L70sd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType Local --outFilterMismatchNmax 2 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_2/L70sd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_2/L70sd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_2/L70sd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_2/L70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_3/70nsd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType Local --outFilterMismatchNmax 3 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_3/70nsd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_3/70nsd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_3/70nsd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_3/70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_3/70sd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType Local --outFilterMismatchNmax 3 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_3/70sd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_3/70sd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_3/70sd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_3/70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_3/L70nsd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType Local --outFilterMismatchNmax 3 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_3/L70nsd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_3/L70nsd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_3/L70nsd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_3/L70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_3/L70sd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType Local --outFilterMismatchNmax 3 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_3/L70sd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_3/L70sd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_3/L70sd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_3/L70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_10/70nsd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType Local --outFilterMismatchNmax 10 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_10/70nsd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_10/70nsd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_10/70nsd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_10/70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_10/70sd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType Local --outFilterMismatchNmax 10 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_10/70sd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_10/70sd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_10/70sd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_10/70sd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_10/L70nsd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType Local --outFilterMismatchNmax 10 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_10/L70nsd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_10/L70nsd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_10/L70nsd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_10/L70nsd\_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /home/ngobet/projects/BXD/references/BXD70\_nonrandomized\_genotypesandimputed/star_paternal\_withannotation --outFileNamePrefix /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_10/L70sd\_ --outSAMtype BAM Unsorted --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --quantMode GeneCounts --scoreDelOpen -2 --scoreInsOpen -2 --alignIntronMax 1 --alignEndsType Local --outFilterMismatchNmax 10 > /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_10/L70sd\_align_default.out 2> /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_10/L70sd\_align.err; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_10/L70sd\_*.bam; rm /mnt/md0/BXD/MappingEvaluation/genotypesandimputed\_withannotation\_Local\_1\_10/L70sd\_SJ.out.tab
