#DATE: Di Mai 26 19:45:21 CEST 2020
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/100nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/100nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/100nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/100nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/100nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/100nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/100nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/100nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/100nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/100sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/100sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/100sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/100sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/100sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/100sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/100sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/100sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/100sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/101nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/101nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/101nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/101nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/101nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/101nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/101nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/101nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/101nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/101sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/101sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/101sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/101sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/101sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/101sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/101sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/101sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/101sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/103nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/103nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/103nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/103nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/103nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/103nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/103nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/103nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/103nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/103sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/103sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/103sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/103sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/103sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/103sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/103sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/103sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/103sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/43nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/43nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/43nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/43nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/43nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/43nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/43nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/43nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/43nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/43sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/43sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/43sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/43sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/43sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/43sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/43sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/43sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/43sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/44nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/44nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/44nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/44nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/44nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/44nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/44nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/44nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/44nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/44sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/44sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/44sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/44sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/44sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/44sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/44sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/44sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/44sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/45nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/45nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/45nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/45nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/45nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/45nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/45nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/45nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/45nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/45sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/45sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/45sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/45sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/45sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/45sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/45sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/45sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/45sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/48nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/48nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/48nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/48nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/48nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/48nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/48nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/48nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/48nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/48sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/48sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/48sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/48sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/48sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/48sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/48sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/48sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/48sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/49nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/49nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/49nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/49nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/49nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/49nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/49nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/49nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/49nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/49sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/49sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/49sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/49sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/49sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/49sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/49sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/49sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/49sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/50nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/50nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/50nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/50nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/50nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/50nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/50nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/50nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/50nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/50sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/50sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/50sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/50sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/50sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/50sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/50sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/50sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/50sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/51nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/51nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/51nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/51nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/51nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/51nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/51nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/51nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/51nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/51sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/51sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/51sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/51sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/51sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/51sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/51sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/51sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/51sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/55nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/55nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/55nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/55nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/55nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/55nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/55nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/55nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/55nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/55sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/55sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/55sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/55sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/55sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/55sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/55sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/55sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/55sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/56nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/56nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/56nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/56nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/56nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/56nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/56nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/56nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/56nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/56sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/56sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/56sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/56sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/56sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/56sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/56sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/56sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/56sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/61nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/61nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/61nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/61nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/61nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/61nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/61nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/61nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/61nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/61sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/61sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/61sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/61sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/61sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/61sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/61sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/61sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/61sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/64nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/64nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/64nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/64nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/64nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/64nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/64nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/64nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/64nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/64sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/64sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/64sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/64sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/64sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/64sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/64sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/64sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/64sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/65nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/65nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/65nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/65nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/65nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/65nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/65nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/65nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/65nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/65sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/65sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/65sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/65sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/65sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/65sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/65sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/65sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/65sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/66nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/66nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/66nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/66nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/66nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/66nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/66nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/66nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/66nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/66sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/66sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/66sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/66sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/66sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/66sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/66sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/66sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/66sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/67nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/67nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/67nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/67nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/67nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/67nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/67nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/67nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/67nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/67sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/67sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/67sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/67sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/67sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/67sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/67sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/67sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/67sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/70nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/70nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/70nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/70nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/70nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/70nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/70nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/70nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/70sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/70sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/70sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/70sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/70sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/70sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/70sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/70sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/71nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/71nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/71nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/71nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/71nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/71nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/71nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/71nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/71nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/71sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/71sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/71sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/71sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/71sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/71sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/71sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/71sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/71sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/73nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/73nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/73nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/73nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/73nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/73nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/73nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/73nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/73nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/73sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/73sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/73sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/73sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/73sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/73sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/73sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/73sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/73sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/75nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/75nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/75nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/75nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/75nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/75nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/75nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/75nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/75nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/75sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/75sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/75sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/75sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/75sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/75sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/75sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/75sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/75sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/79nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/79nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/79nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/79nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/79nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/79nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/79nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/79nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/79nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/79sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/79sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/79sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/79sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/79sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/79sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/79sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/79sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/79sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/81nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/81nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/81nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/81nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/81nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/81nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/81nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/81nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/81nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/81sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/81sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/81sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/81sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/81sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/81sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/81sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/81sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/81sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/83nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/83nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/83nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/83nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/83nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/83nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/83nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/83nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/83nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/83sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/83sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/83sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/83sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/83sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/83sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/83sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/83sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/83sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/84nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/84nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/84nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/84nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/84nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/84nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/84nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/84nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/84nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/84sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/84sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/84sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/84sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/84sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/84sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/84sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/84sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/84sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/85nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/85nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/85nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/85nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/85nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/85nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/85nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/85nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/85nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/85sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/85sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/85sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/85sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/85sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/85sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/85sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/85sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/85sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/87nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/87nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/87nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/87nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/87nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/87nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/87nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/87nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/87nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/87sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/87sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/87sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/87sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/87sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/87sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/87sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/87sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/87sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/89nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/89nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/89nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/89nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/89nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/89nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/89nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/89nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/89nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/89sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/89sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/89sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/89sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/89sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/89sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/89sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/89sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/89sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/90nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/90nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/90nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/90nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/90nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/90nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/90nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/90nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/90nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/90sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/90sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/90sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/90sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/90sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/90sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/90sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/90sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/90sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/95nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/95nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/95nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/95nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/95nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/95nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/95nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/95nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/95nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/95sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/95sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/95sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/95sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/95sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/95sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/95sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/95sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/95sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/96nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/96nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/96nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/96nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/96nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/96nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/96nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/96nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/96nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/96sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/96sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/96sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/96sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/96sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/96sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/96sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/96sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/96sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/97nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/97nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/97nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/97nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/97nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/97nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/97nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/97nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/97nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/97sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/97sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/97sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/97sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/97sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/97sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/97sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/97sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/97sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/98nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/98nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/98nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/98nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/98nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/98nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/98nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/98nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/98nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/98sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/98sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/98sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/98sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/98sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/98sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/98sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/98sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/98sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B61nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/B61nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B61nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B61nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B61nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B61nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B61nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B61nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B61nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B61sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/B61sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B61sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B61sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B61sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B61sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B61sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B61sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B61sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B62nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/B62nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B62nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B62nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B62nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B62nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B62nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B62nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B62nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B62sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/B62sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B62sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B62sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B62sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B62sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B62sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B62sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B62sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/BDnsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/BDnsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/BDnsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/BDnsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/BDnsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/BDnsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/BDnsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/BDnsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/BDnsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/BDsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/BDsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/BDsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/BDsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/BDsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/BDsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/BDsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/BDsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/BDsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB1nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/DB1nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB1nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB1nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB1nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB1nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB1nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB1nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB1nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB1sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/DB1sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB1sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB1sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB1sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB1sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB1sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB1sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB1sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB2nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/DB2nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB2nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB2nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB2nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB2nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB2nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB2nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB2nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB2sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/DB2sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB2sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB2sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB2sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB2sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB2sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB2sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB2sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DBnsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/DBnsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DBnsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DBnsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DBnsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DBnsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DBnsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DBnsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DBnsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DBsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/DBsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DBsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DBsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DBsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DBsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DBsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DBsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DBsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L100nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L100nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L100nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L100nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L100nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L100nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L100nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L100nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L100nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L100sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L100sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L100sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L100sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L100sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L100sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L100sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L100sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L100sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L101nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L101nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L101nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L101nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L101nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L101nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L101nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L101nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L101nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L101sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L101sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L101sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L101sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L101sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L101sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L101sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L101sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L101sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L103nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L103nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L103nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L103nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L103nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L103nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L103nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L103nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L103nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L103sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L103sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L103sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L103sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L103sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L103sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L103sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L103sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L103sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L43nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L43nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L43nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L43nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L43nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L43nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L43nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L43nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L43nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L43sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L43sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L43sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L43sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L43sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L43sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L43sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L43sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L43sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L44nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L44nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L44nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L44nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L44nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L44nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L44nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L44nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L44nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L44sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L44sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L44sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L44sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L44sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L44sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L44sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L44sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L44sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L45nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L45nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L45nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L45nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L45nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L45nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L45nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L45nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L45nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L45sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L45sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L45sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L45sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L45sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L45sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L45sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L45sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L45sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L48nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L48nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L48nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L48nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L48nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L48nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L48nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L48nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L48nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L48sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L48sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L48sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L48sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L48sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L48sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L48sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L48sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L48sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L49nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L49nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L49nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L49nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L49nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L49nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L49nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L49nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L49nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L49sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L49sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L49sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L49sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L49sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L49sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L49sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L49sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L49sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L50nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L50nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L50nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L50nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L50nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L50nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L50nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L50nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L50nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L50sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L50sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L50sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L50sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L50sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L50sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L50sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L50sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L50sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L51nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L51nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L51nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L51nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L51nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L51nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L51nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L51nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L51nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L51sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L51sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L51sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L51sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L51sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L51sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L51sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L51sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L51sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L55nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L55nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L55nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L55nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L55nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L55nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L55nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L55nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L55nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L55sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L55sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L55sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L55sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L55sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L55sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L55sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L55sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L55sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L56nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L56nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L56nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L56nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L56nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L56nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L56nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L56nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L56nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L56sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L56sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L56sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L56sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L56sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L56sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L56sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L56sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L56sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L61nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L61nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L61nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L61nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L61nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L61nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L61nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L61nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L61nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L61sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L61sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L61sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L61sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L61sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L61sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L61sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L61sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L61sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L64nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L64nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L64nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L64nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L64nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L64nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L64nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L64nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L64nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L64sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L64sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L64sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L64sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L64sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L64sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L64sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L64sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L64sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L65nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L65nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L65nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L65nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L65nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L65nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L65nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L65nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L65nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L65sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L65sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L65sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L65sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L65sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L65sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L65sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L65sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L65sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L66nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L66nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L66nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L66nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L66nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L66nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L66nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L66nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L66nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L66sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L66sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L66sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L66sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L66sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L66sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L66sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L66sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L66sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L67nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L67nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L67nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L67nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L67nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L67nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L67nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L67nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L67nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L67sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L67sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L67sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L67sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L67sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L67sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L67sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L67sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L67sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L70nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L70nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L70nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L70nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L70nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L70nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L70nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L70nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L70sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L70sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L70sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L70sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L70sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L70sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L70sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L70sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L71nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L71nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L71nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L71nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L71nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L71nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L71nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L71nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L71nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L71sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L71sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L71sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L71sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L71sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L71sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L71sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L71sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L71sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L73nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L73nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L73nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L73nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L73nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L73nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L73nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L73nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L73nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L73sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L73sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L73sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L73sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L73sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L73sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L73sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L73sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L73sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L75nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L75nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L75nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L75nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L75nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L75nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L75nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L75nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L75nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L75sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L75sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L75sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L75sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L75sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L75sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L75sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L75sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L75sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L79nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L79nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L79nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L79nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L79nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L79nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L79nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L79nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L79nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L79sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L79sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L79sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L79sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L79sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L79sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L79sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L79sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L79sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L81nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L81nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L81nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L81nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L81nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L81nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L81nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L81nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L81nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L81sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L81sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L81sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L81sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L81sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L81sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L81sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L81sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L81sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L83nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L83nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L83nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L83nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L83nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L83nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L83nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L83nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L83nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L83sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L83sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L83sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L83sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L83sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L83sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L83sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L83sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L83sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L84nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L84nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L84nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L84nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L84nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L84nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L84nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L84nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L84nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L84sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L84sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L84sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L84sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L84sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L84sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L84sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L84sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L84sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L85nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L85nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L85nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L85nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L85nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L85nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L85nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L85nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L85nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L85sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L85sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L85sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L85sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L85sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L85sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L85sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L85sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L85sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L87nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L87nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L87nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L87nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L87nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L87nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L87nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L87nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L87nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L87sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L87sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L87sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L87sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L87sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L87sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L87sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L87sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L87sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L89nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L89nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L89nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L89nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L89nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L89nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L89nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L89nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L89nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L89sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L89sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L89sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L89sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L89sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L89sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L89sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L89sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L89sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L90nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L90nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L90nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L90nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L90nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L90nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L90nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L90nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L90nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L90sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L90sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L90sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L90sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L90sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L90sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L90sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L90sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L90sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L95nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L95nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L95nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L95nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L95nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L95nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L95nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L95nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L95nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L95sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L95sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L95sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L95sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L95sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L95sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L95sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L95sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L95sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L96nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L96nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L96nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L96nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L96nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L96nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L96nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L96nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L96nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L96sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L96sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L96sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L96sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L96sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L96sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L96sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L96sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L96sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L97nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L97nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L97nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L97nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L97nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L97nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L97nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L97nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L97nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L97sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L97sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L97sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L97sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L97sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L97sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L97sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L97sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L97sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L98nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L98nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L98nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L98nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L98nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L98nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L98nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L98nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L98nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L98sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L98sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L98sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L98sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L98sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L98sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L98sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L98sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L98sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB61nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/LB61nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB61nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB61nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB61nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB61nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB61nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB61nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB61nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB61sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/LB61sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB61sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB61sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB61sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB61sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB61sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB61sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB61sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB62nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/LB62nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB62nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB62nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB62nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB62nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB62nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB62nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB62nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB62sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/LB62sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB62sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB62sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB62sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB62sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB62sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB62sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB62sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LBDnsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/LBDnsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LBDnsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LBDnsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LBDnsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LBDnsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LBDnsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LBDnsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LBDnsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LBDsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/LBDsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LBDsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LBDsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LBDsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LBDsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LBDsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LBDsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LBDsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB1nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/LDB1nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB1nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB1nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB1nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB1nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB1nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB1nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB1nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB1sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/LDB1sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB1sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB1sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB1sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB1sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB1sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB1sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB1sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB2nsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/LDB2nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB2nsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB2nsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB2nsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB2nsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB2nsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB2nsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB2nsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB2sd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/LDB2sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB2sd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB2sd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB2sd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB2sd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB2sd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB2sd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB2sd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDBnsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/LDBnsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDBnsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDBnsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDBnsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDBnsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDBnsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDBnsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDBnsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withoutannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDBsd\_restrictive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/LDBsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --scoreDelOpen -40 --scoreInsOpen -40 --alignIntronMax 1 --alignEndsType EndToEnd --outFilterMismatchNmax 0 > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDBsd\_restrictive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDBsd\_restrictive_align.err; htseq-count -f bam -q -s reverse -t exon -m union /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDBsd\_restrictive_Aligned.out.bam /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/paternal.gtf 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDBsd\_htseqcount.err 1> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDBsd\_restrictive_ReadsPerGene.out.tab; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDBsd\_restrictive*.bam; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDBsd\_restrictive*SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/100nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/100nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/100nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/100nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/100nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/100nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/100sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/100sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/100sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/100sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/100sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/100sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/101nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/101nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/101nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/101nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/101nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/101nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/101sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/101sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/101sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/101sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/101sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/101sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/103nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/103nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/103nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/103nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/103nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/103nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/103sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/103sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/103sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/103sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/103sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/103sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/43nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/43nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/43nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/43nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/43nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/43nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/43sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/43sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/43sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/43sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/43sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/43sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/44nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/44nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/44nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/44nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/44nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/44nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/44sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/44sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/44sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/44sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/44sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/44sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/45nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/45nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/45nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/45nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/45nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/45nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/45sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/45sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/45sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/45sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/45sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/45sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/48nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/48nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/48nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/48nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/48nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/48nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/48sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/48sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/48sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/48sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/48sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/48sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/49nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/49nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/49nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/49nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/49nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/49nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/49sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/49sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/49sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/49sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/49sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/49sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/50nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/50nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/50nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/50nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/50nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/50nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/50sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/50sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/50sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/50sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/50sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/50sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/51nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/51nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/51nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/51nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/51nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/51nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/51sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/51sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/51sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/51sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/51sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/51sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/55nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/55nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/55nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/55nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/55nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/55nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/55sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/55sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/55sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/55sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/55sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/55sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/56nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/56nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/56nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/56nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/56nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/56nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/56sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/56sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/56sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/56sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/56sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/56sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/61nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/61nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/61nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/61nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/61nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/61nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/61sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/61sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/61sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/61sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/61sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/61sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/64nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/64nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/64nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/64nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/64nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/64nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/64sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/64sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/64sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/64sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/64sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/64sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/65nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/65nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/65nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/65nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/65nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/65nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/65sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/65sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/65sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/65sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/65sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/65sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/66nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/66nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/66nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/66nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/66nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/66nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/66sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/66sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/66sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/66sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/66sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/66sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/67nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/67nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/67nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/67nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/67nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/67nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/67sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/67sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/67sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/67sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/67sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/67sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/70nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/70nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/70nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/70nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/70nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/70sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/70sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/70sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/70sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/70sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/70sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/71nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/71nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/71nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/71nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/71nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/71nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/71sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/71sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/71sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/71sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/71sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/71sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/73nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/73nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/73nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/73nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/73nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/73nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/73sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/73sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/73sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/73sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/73sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/73sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/75nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/75nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/75nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/75nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/75nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/75nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/75sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/75sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/75sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/75sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/75sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/75sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/79nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/79nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/79nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/79nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/79nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/79nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/79sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/79sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/79sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/79sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/79sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/79sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/81nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/81nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/81nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/81nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/81nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/81nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/81sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/81sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/81sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/81sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/81sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/81sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/83nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/83nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/83nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/83nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/83nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/83nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/83sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/83sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/83sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/83sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/83sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/83sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/84nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/84nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/84nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/84nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/84nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/84nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/84sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/84sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/84sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/84sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/84sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/84sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/85nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/85nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/85nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/85nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/85nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/85nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/85sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/85sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/85sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/85sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/85sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/85sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/87nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/87nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/87nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/87nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/87nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/87nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/87sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/87sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/87sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/87sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/87sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/87sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/89nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/89nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/89nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/89nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/89nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/89nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/89sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/89sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/89sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/89sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/89sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/89sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/90nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/90nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/90nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/90nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/90nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/90nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/90sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/90sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/90sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/90sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/90sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/90sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/95nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/95nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/95nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/95nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/95nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/95nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/95sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/95sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/95sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/95sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/95sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/95sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/96nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/96nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/96nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/96nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/96nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/96nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/96sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/96sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/96sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/96sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/96sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/96sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/97nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/97nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/97nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/97nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/97nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/97nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/97sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/97sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/97sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/97sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/97sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/97sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/98nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/98nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/98nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/98nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/98nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/98nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/98sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/98sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/98sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/98sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/98sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/98sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B61nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/B61nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B61nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B61nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B61nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B61nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B61sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/B61sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B61sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B61sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B61sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B61sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B62nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/B62nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B62nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B62nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B62nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B62nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B62sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/B62sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B62sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B62sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B62sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/B62sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/BDnsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/BDnsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/BDnsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/BDnsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/BDnsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/BDnsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/BDsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/BDsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/BDsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/BDsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/BDsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/BDsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB1nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/DB1nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB1nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB1nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB1nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB1nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB1sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/DB1sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB1sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB1sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB1sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB1sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB2nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/DB2nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB2nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB2nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB2nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB2nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB2sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/DB2sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB2sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB2sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB2sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DB2sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DBnsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/DBnsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DBnsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DBnsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DBnsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DBnsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DBsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/DBsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DBsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DBsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DBsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/DBsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L100nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L100nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L100nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L100nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L100nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L100nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L100sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L100sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L100sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L100sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L100sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L100sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L101nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L101nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L101nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L101nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L101nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L101nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L101sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L101sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L101sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L101sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L101sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L101sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L103nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L103nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L103nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L103nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L103nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L103nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L103sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L103sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L103sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L103sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L103sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L103sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L43nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L43nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L43nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L43nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L43nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L43nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L43sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L43sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L43sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L43sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L43sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L43sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L44nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L44nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L44nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L44nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L44nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L44nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L44sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L44sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L44sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L44sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L44sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L44sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L45nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L45nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L45nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L45nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L45nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L45nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L45sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L45sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L45sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L45sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L45sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L45sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L48nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L48nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L48nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L48nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L48nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L48nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L48sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L48sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L48sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L48sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L48sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L48sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L49nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L49nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L49nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L49nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L49nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L49nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L49sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L49sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L49sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L49sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L49sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L49sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L50nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L50nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L50nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L50nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L50nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L50nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L50sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L50sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L50sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L50sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L50sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L50sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L51nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L51nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L51nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L51nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L51nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L51nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L51sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L51sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L51sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L51sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L51sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L51sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L55nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L55nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L55nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L55nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L55nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L55nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L55sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L55sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L55sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L55sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L55sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L55sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L56nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L56nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L56nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L56nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L56nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L56nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L56sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L56sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L56sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L56sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L56sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L56sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L61nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L61nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L61nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L61nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L61nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L61nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L61sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L61sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L61sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L61sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L61sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L61sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L64nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L64nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L64nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L64nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L64nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L64nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L64sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L64sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L64sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L64sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L64sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L64sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L65nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L65nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L65nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L65nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L65nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L65nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L65sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L65sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L65sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L65sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L65sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L65sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L66nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L66nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L66nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L66nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L66nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L66nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L66sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L66sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L66sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L66sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L66sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L66sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L67nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L67nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L67nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L67nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L67nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L67nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L67sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L67sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L67sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L67sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L67sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L67sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L70nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L70nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L70nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L70nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L70nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L70sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L70sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L70sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L70sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L70sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L70sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L71nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L71nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L71nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L71nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L71nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L71nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L71sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L71sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L71sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L71sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L71sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L71sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L73nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L73nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L73nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L73nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L73nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L73nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L73sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L73sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L73sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L73sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L73sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L73sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L75nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L75nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L75nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L75nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L75nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L75nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L75sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L75sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L75sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L75sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L75sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L75sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L79nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L79nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L79nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L79nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L79nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L79nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L79sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L79sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L79sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L79sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L79sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L79sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L81nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L81nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L81nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L81nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L81nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L81nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L81sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L81sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L81sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L81sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L81sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L81sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L83nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L83nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L83nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L83nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L83nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L83nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L83sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L83sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L83sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L83sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L83sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L83sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L84nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L84nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L84nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L84nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L84nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L84nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L84sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L84sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L84sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L84sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L84sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L84sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L85nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L85nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L85nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L85nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L85nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L85nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L85sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L85sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L85sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L85sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L85sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L85sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L87nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L87nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L87nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L87nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L87nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L87nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L87sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L87sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L87sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L87sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L87sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L87sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L89nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L89nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L89nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L89nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L89nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L89nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L89sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L89sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L89sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L89sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L89sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L89sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L90nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L90nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L90nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L90nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L90nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L90nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L90sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L90sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L90sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L90sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L90sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L90sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L95nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L95nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L95nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L95nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L95nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L95nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L95sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L95sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L95sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L95sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L95sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L95sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L96nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L96nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L96nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L96nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L96nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L96nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L96sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L96sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L96sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L96sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L96sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L96sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L97nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L97nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L97nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L97nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L97nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L97nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L97sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L97sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L97sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L97sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L97sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L97sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L98nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L98nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L98nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L98nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L98nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L98nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L98sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/L98sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L98sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L98sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L98sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/L98sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB61nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/LB61nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB61nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB61nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB61nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB61nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB61sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/LB61sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB61sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB61sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB61sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB61sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB62nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/LB62nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB62nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB62nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB62nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB62nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB62sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/LB62sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB62sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB62sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB62sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LB62sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LBDnsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/LBDnsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LBDnsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LBDnsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LBDnsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LBDnsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LBDsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/LBDsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LBDsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LBDsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LBDsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LBDsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB1nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/LDB1nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB1nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB1nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB1nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB1nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB1sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/LDB1sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB1sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB1sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB1sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB1sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB2nsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/LDB2nsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB2nsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB2nsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB2nsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB2nsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB2sd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/LDB2sd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB2sd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB2sd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB2sd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDB2sd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDBnsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/LDBnsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDBnsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDBnsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDBnsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDBnsd\_permissive_SJ.out.tab
/software/Alignment/STAR/STAR-2.7.0e/bin/STAR --genomeDir /mnt/nas/BXD/data/PersonalizedReferences/DBA_2J_nonrandomized_indelsSNVs/star_paternal_withannotation --outFileNamePrefix /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDBsd\_permissive_ --runThreadN 2 --readFilesCommand zcat --readFilesIn /mnt/nas/BXD/data/transcriptome/RNAseq/filtered/LDBsd.fastq.gz --outSAMtype BAM Unsorted --outStd Log --quantMode GeneCounts > /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDBsd\_permissive_align.out 2> /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDBsd\_permissive_align.err; rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDBsd\_permissive_Aligned.out.bam;	rm /mnt/md0/BXD/DBA_2J_nonrandomized_indelsSNVs/LDBsd\_permissive_SJ.out.tab
